From 6ceffc2725ed4146510436938ffda7dc0acc6c5d Mon Sep 17 00:00:00 2001
From: "Luis Gustavo S. Barreto" <gustavo@ossystems.com.br>
Date: Tue, 3 May 2016 18:36:14 +0000
Subject: [PATCH] Remove freebsd compilation support

Signed-off-by: Luis Gustavo S. Barreto <gustavo@ossystems.com.br>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5976e69..0711b11 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,9 +1,5 @@
 cmake_minimum_required(VERSION 2.8.11 FATAL_ERROR)
 
-#freebsd support
-include_directories(SYSTEM /usr/local/include)
-SET(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES} /usr/local/include)
-
 #macports support
 IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   SET(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES} /opt/local/include)
-- 
2.1.4

