From 7dbeaa5509237f3eba7152035d4eb9620c6abaee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 15 Jan 2014 18:54:06 +0100
Subject: [PATCH] client/CMakeLists.txt: avoid dragging in host libdir
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/client/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/CMakeLists.txt b/src/client/CMakeLists.txt
index 3bdb0fd..c2fb4a6 100644
--- a/src/client/CMakeLists.txt
+++ b/src/client/CMakeLists.txt
@@ -107,7 +107,7 @@ add_executable(hawaii-desktop-shell ${SOURCES} ${RESOURCES})
 qt5_use_modules(hawaii-desktop-shell Quick Widgets)
 set_target_properties(hawaii-desktop-shell PROPERTIES
     COMPILE_FLAGS "-rdynamic"
-    LINK_FLAGS "-L${WaylandClient_LIBDIR} -L${POLKITQT-1_LIB_DIR}"
+    LINK_FLAGS "-L${POLKITQT-1_LIB_DIR}"
 )
 target_link_libraries(hawaii-desktop-shell
     ${WaylandClient_LIBRARIES}
-- 
1.8.3.1

