From d5c93a4838c8283b0a13ac1ca2395118cefe7e94 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 3 Dec 2014 16:02:52 +0100
Subject: [PATCH] install icemaker - we need the native one for cross
 compilation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/kjs/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/kjs/CMakeLists.txt b/src/kjs/CMakeLists.txt
index d8406ea..3286334 100644
--- a/src/kjs/CMakeLists.txt
+++ b/src/kjs/CMakeLists.txt
@@ -67,6 +67,7 @@ endif()
    add_executable(icemaker ${icemaker_SRCS})
    ecm_mark_nongui_executable(icemaker)
    target_include_directories(icemaker PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/..")
+   install(TARGETS icemaker EXPORT KF5JSTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
 
    # get the name of the generated wrapper script (which sets up LD_LIBRARY_PATH)
    #get_target_property(ICEMAKER_EXECUTABLE icemaker WRAPPER_SCRIPT)
-- 
1.8.3.1

