From f2b5981d848d700212fc30d719c30db6d57cd2e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 2 Feb 2015 14:28:10 +0100
Subject: [PATCH] fix build for qtbase without session management
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 main.cpp      |  8 +++++++-
 workspace.cpp | 11 ++++++++++-
 2 files changed, 17 insertions(+), 2 deletions(-)

diff --git a/main.cpp b/main.cpp
index 44d8c49..0cb9e44 100644
--- a/main.cpp
+++ b/main.cpp
@@ -360,7 +360,13 @@ void Application::createWorkspace()
     // critical startup section where x errors cause kwin to abort.
 
     // create workspace.
-    (void) new Workspace(isSessionRestored());
+    (void) new Workspace(
+#ifndef QT_NO_SESSIONMANAGER
+        isSessionRestored()
+#else
+        false
+#endif
+      );
 }
 
 void Application::createAtoms()
diff --git a/workspace.cpp b/workspace.cpp
index 7cbe81e..66c603b 100644
--- a/workspace.cpp
+++ b/workspace.cpp
@@ -277,8 +277,11 @@ void Workspace::init()
     // Extra NETRootInfo instance in Client mode is needed to get the values of the properties
     NETRootInfo client_info(connection(), NET::ActiveWindow | NET::CurrentDesktop);
     int initial_desktop;
+#ifndef QT_NO_SESSIONMANAGER
     if (!qApp->isSessionRestored())
+#endif // QT_NO_SESSIONMANAGER
         initial_desktop = client_info.currentDesktop();
+#ifndef QT_NO_SESSIONMANAGER
     else {
 #if KWIN_QT5_PORTING
         KConfigGroup group(kapp->sessionConfig(), "Session");
@@ -287,6 +290,7 @@ void Workspace::init()
         initial_desktop = 1;
 #endif
     }
+#endif // QT_NO_SESSIONMANAGER
     if (!VirtualDesktopManager::self()->setCurrent(initial_desktop))
         VirtualDesktopManager::self()->setCurrent(1);
 
@@ -307,7 +311,9 @@ void Workspace::init()
     active_client = NULL;
     rootInfo->setActiveWindow(None);
     focusToNull();
+#ifndef QT_NO_SESSIONMANAGER
     if (!qApp->isSessionRestored())
+#endif // QT_NO_SESSIONMANAGER
         ++block_focus; // Because it will be set below
 
     {
@@ -372,7 +378,10 @@ void Workspace::init()
     } // End updates blocker block
 
     Client* new_active_client = NULL;
-    if (!qApp->isSessionRestored()) {
+#ifndef QT_NO_SESSIONMANAGER
+    if (!qApp->isSessionRestored())
+#endif
+    {
         --block_focus;
         new_active_client = findClient(Predicate::WindowMatch, client_info.activeWindow());
     }
-- 
1.8.3.1

