From 670bb48076453b98c2d6ef5492fb1ea4cfbdded0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 1 Jun 2015 14:27:20 +0200
Subject: [PATCH] add egl flags for compiling
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt                | 4 ++++
 libkwineffects/CMakeLists.txt | 3 +++
 2 files changed, 7 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7866d2c..c678bd4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -562,6 +562,10 @@ set_target_properties(kwin PROPERTIES
    SOVERSION ${PROJECT_VERSION_MAJOR}
    )
 
+if(KWIN_HAVE_EGL)
+    target_compile_options(kwin PUBLIC ${EGL_DEFINITIONS})
+endif()
+
 target_link_libraries(kwin ${kwinLibs})
 generate_export_header(kwin EXPORT_FILE_NAME kwin_export.h)
 
diff --git a/libkwineffects/CMakeLists.txt b/libkwineffects/CMakeLists.txt
index 1797ed5..2d71cb8 100644
--- a/libkwineffects/CMakeLists.txt
+++ b/libkwineffects/CMakeLists.txt
@@ -92,6 +92,9 @@ macro( KWIN4_ADD_GLUTILS_BACKEND name glinclude )
         VERSION ${KWINEFFECTS_VERSION_STRING}
         SOVERSION ${KWINEFFECTS_SOVERSION}
     )
+    if(KWIN_HAVE_EGL)
+        target_compile_options(${name} PUBLIC ${EGL_DEFINITIONS})
+    endif()
     target_link_libraries(${name} PUBLIC ${ARGN})
 
     install(TARGETS ${name} EXPORT kdeworkspaceLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
-- 
1.9.3

