--- a/apps/KMixApp.cpp	2016-06-07 16:48:58.967601212 +0200
+++ b/apps/KMixApp.cpp	2016-06-07 16:49:47.720048427 +0200
@@ -80,7 +80,11 @@
 	 */
 	creationLock.lock();
 
+#ifndef QT_NO_SESSIONMANAGER
 	bool restore = isSessionRestored(); // && KMainWindow::canBeRestored(0);
+#else
+	bool restore = false;
+#endif
 	kDebug() << "Starting KMix using kepvisibility=" << hasArgKeepvisibility << ", failsafe=" << reset << ", sessionRestore=" << restore;
 	int createCount = 0;
 	if (restore)
--- a/apps/kmix.cpp	2016-06-07 16:50:51.464632922 +0200
+++ b/apps/kmix.cpp	2016-06-07 16:50:24.112382155 +0200
@@ -116,9 +116,11 @@
 		recreateGUI(false, QString(), true, reset);
 	}
 
+#ifndef QT_NO_SESSIONMANAGER
 	if (!kapp->isSessionRestored() ) // done by the session manager otherwise
+#endif
 	setInitialSize();
 
 	fixConfigAfterRead();
