From d3288c56543300b7cb115e9fe8adde3a89391116 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 9 Dec 2014 06:25:44 +0100
Subject: [PATCH] kdesud.cpp: check HAVE_X11 properly
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/kdesud/kdesud.cpp | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/kdesud/kdesud.cpp b/src/kdesud/kdesud.cpp
index 1ca5fdf..f991727 100644
--- a/src/kdesud/kdesud.cpp
+++ b/src/kdesud/kdesud.cpp
@@ -73,7 +73,7 @@
 #include "repo.h"
 #include "handler.h"
 
-#ifdef HAVE_X11
+#if HAVE_X11
 #include <X11/X.h>
 #include <X11/Xlib.h>
 #endif
@@ -94,7 +94,7 @@ using namespace KDESu;
 Repository *repo;
 QString Version(QStringLiteral("1.01"));
 QByteArray sock;
-#ifdef HAVE_X11
+#if HAVE_X11
 Display *x11Display;
 #endif
 int pipeOfDeath[2];
@@ -107,7 +107,7 @@ void kdesud_cleanup()
 
 // Borrowed from kdebase/kaudio/kaudioserver.cpp
 
-#ifdef HAVE_X11
+#if HAVE_X11
 extern "C" int xio_errhandler(Display *);
 
 int xio_errhandler(Display *)
@@ -307,7 +307,7 @@ int main(int argc, char *argv[])
     if (pid)
         _exit(0);
 
-#ifdef HAVE_X11
+#if HAVE_X11
     // Make sure we exit when the display gets closed.
     int x11Fd = initXconnection();
     maxfd = qMax(maxfd, x11Fd);
@@ -344,7 +344,7 @@ int main(int argc, char *argv[])
     FD_ZERO(&active_fds);
     FD_SET(sockfd, &active_fds);
     FD_SET(pipeOfDeath[0], &active_fds);
-#ifdef HAVE_X11
+#if HAVE_X11
     if (x11Fd != -1)
         FD_SET(x11Fd, &active_fds);
 #endif
@@ -352,7 +352,7 @@ int main(int argc, char *argv[])
     while (1)
     {
         tmp_fds = active_fds;
-#ifdef HAVE_X11
+#if HAVE_X11
         if(x11Display)
             XFlush(x11Display);
 #endif
@@ -396,7 +396,7 @@ int main(int argc, char *argv[])
                 while(result > 0);
             }
 
-#ifdef HAVE_X11
+#if HAVE_X11
             if (i == x11Fd)
             {
                 // Discard X events
-- 
1.8.3.1

