From 40fff6e760920d6b31dc98dbcbde081845ac26b0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 26 Aug 2015 22:31:35 +0200
Subject: [PATCH 1/2] Makefile: use pkgconfig to find libpcre
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e66f322..d9737f7 100644
--- a/Makefile
+++ b/Makefile
@@ -39,8 +39,8 @@ endif
 # Enables support for PCRE regular expression pattern in window conditions
 ifeq "$(NO_REGEX_PCRE)" ""
   CFG += -DCONFIG_REGEX_PCRE
-  LIBS += $(shell pcre-config --libs)
-  INCS += $(shell pcre-config --cflags)
+  LIBS += $(shell pkg-config --libs libpcre)
+  INCS += $(shell pkg-config --cflags libpcre)
   # Enables JIT support in libpcre
   ifeq "$(NO_REGEX_PCRE_JIT)" ""
     CFG += -DCONFIG_REGEX_PCRE_JIT
-- 
2.1.0

