From cad9d387c113c0ffeff8005a7131b6a545ab86bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 21 Aug 2016 21:25:07 +0200
Subject: [PATCH] configure.ac: do not der absolute default paths
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6812734..0928946 100644
--- a/configure.ac
+++ b/configure.ac
@@ -114,8 +114,6 @@ AC_ARG_ENABLE(stacktrace,
   [ac_stacktrace="$enableval"])
 
 
-# Standard installation base dirs.
-ac_with_paths="/usr /usr/local"
 
 # Set for alternate Qt4/5 installation dir.
 AC_ARG_WITH(qt4,
-- 
2.5.5

