From bd1fa7715d677db3bff8bfed6f751edfab88543d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 8 Dec 2014 16:51:28 +0100
Subject: [PATCH] return WAYLAND_DISPLAY if all known options fail
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/kdeinit/kinit.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/kdeinit/kinit.cpp b/src/kdeinit/kinit.cpp
index 3c3c913..346fea7 100644
--- a/src/kdeinit/kinit.cpp
+++ b/src/kdeinit/kinit.cpp
@@ -122,6 +122,8 @@ static QByteArray displayEnvVarName()
     return "MAC_DISPLAY";
 #elif defined(Q_OS_WIN)
     return "WIN_DISPLAY";
+#else
+    return "WAYLAND_DISPLAY";
 #endif
 }
 
-- 
1.8.3.1

