From 02ed7d968dc1c4e8140c73bfe8e9becac1e2aad7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 15 Feb 2020 11:16:18 +0100
Subject: [PATCH 1/2] use pkgconfig to find libgcrypt
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8594d35..9c62522 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,8 @@ include(KDEClangFormat)
 include(ECMConfiguredInstall)
 
 find_package(PAM REQUIRED)
-find_package(LibGcrypt 1.5.0 REQUIRED)
+find_package(PkgConfig REQUIRED)
+pkg_search_module(LIBGCRYPT REQUIRED libgcrypt)
 
 find_program(SOCAT_EXECUTABLE socat)
 
-- 
2.21.0

