From 4ed9c206257199eb8ea47fb7abecf993b0c0a23e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 14 Aug 2015 21:53:55 +0200
Subject: [PATCH] config-kwindowsystem.h.cmake: fix path to plugins
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/config-kwindowsystem.h.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config-kwindowsystem.h.cmake b/src/config-kwindowsystem.h.cmake
index 9afe81f..7bc1d03 100644
--- a/src/config-kwindowsystem.h.cmake
+++ b/src/config-kwindowsystem.h.cmake
@@ -10,4 +10,4 @@
 #cmakedefine01 KWINDOWSYSTEM_HAVE_XRENDER
 
 /* Path to xcb plugin */
-#define XCB_PLUGIN_PATH "${CMAKE_INSTALL_PREFIX}/${PLUGIN_INSTALL_DIR}/org.kde.kwindowsystem.platforms/KF5WindowSystemX11Plugin.so"
+#define XCB_PLUGIN_PATH "${PLUGIN_INSTALL_DIR}/kf5/org.kde.kwindowsystem.platforms/KF5WindowSystemX11Plugin.so"
-- 
2.1.0

