From e597c5c8636a269dbe5e664c706e66887479f161 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Tue, 25 Sep 2018 00:44:12 +0200
Subject: [PATCH] do not build tests

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 wscript | 2 --
 1 file changed, 2 deletions(-)

diff --git a/wscript b/wscript
index c08298c..1443ae2 100644
--- a/wscript
+++ b/wscript
@@ -494,8 +494,6 @@ def build(bld):
             bld.options.testcmd = 'node %s'
         if bld.options.enable_examples:
             bld.recurse('examples')
-        if bld.options.enable_tests:
-            bld.recurse('tests')
 
     # pkg-config template
     bld( source = 'aubio.pc.in' )
-- 
2.14.4

