From 6ff4dcce6aa865f96af7273e875c722edbceb83e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Thu, 13 Sep 2018 21:27:40 +0200
Subject: [PATCH] Native: Do not build X11 dependent zest/libzest.so
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Makefile | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Makefile b/Makefile
index 7ce309a..9d38910 100644
--- a/Makefile
+++ b/Makefile
@@ -12,11 +12,6 @@ all:
 	cd deps/pugl         && python2 ./waf
 	cd src/osc-bridge    && make lib
 	cd mruby             && MRUBY_CONFIG=../build_config.rb rake
-	$(CC) -shared -o libzest.so `find mruby/build/host -type f | grep -e "\.o$$" | grep -v bin` ./deps/libnanovg.a \
-		./deps/libnanovg.a \
-		src/osc-bridge/libosc-bridge.a \
-		$(LDFLAGS) -lpcre -luv -lm -lX11 -lGL -lpthread
-	$(CC) $(CFLAGS) test-libversion.c deps/pugl/build/libpugl-0.a -ldl -o zest $(LDFLAGS) -lX11 -lGL -lpthread -I deps/pugl -std=gnu99
 
 osx:
 	ruby ./rebuild-fcache.rb
-- 
2.14.4

