From e977098c48c7cbea7bfc13b059759d02e78e5f55 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 1 Dec 2014 19:14:24 +0100
Subject: [PATCH] align KI18N_MODULE_DIR to proper path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 KF5I18nConfig.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/KF5I18nConfig.cmake.in b/KF5I18nConfig.cmake.in
index 64dc500..7c0c1f6 100644
--- a/KF5I18nConfig.cmake.in
+++ b/KF5I18nConfig.cmake.in
@@ -1,6 +1,6 @@
 @PACKAGE_INIT@
 
-set(KI18N_MODULE_DIR "${PACKAGE_PREFIX_DIR}/@CMAKECONFIG_INSTALL_DIR@")
+set(KI18N_MODULE_DIR "${CMAKE_CURRENT_LIST_DIR}")
 
 include("${CMAKE_CURRENT_LIST_DIR}/KF5I18nTargets.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/KF5I18NMacros.cmake")
-- 
1.8.3.1

