From c8f9d29e0f63e2973ee1ad2c4b0696e0a2d13882 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 25 Mar 2016 15:44:08 +0100
Subject: [PATCH] make kwayland support optional
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt          | 2 +-
 backends/CMakeLists.txt | 5 +++++
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 711f9da..891a4b8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@ set(REQUIRED_QT_VERSION 5.2.0)
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Core DBus Gui Test X11Extras)
 
 # Wayland backend
-find_package(KF5Wayland CONFIG REQUIRED)
+find_package(KF5Wayland CONFIG QUIET)
 add_feature_info("KF5Wayland" KF5Wayland_FOUND "Required for building libkscreen's KWayland backend")
 
 # xrandr backend
diff --git a/backends/CMakeLists.txt b/backends/CMakeLists.txt
index 3563e13..195aaa9 100644
--- a/backends/CMakeLists.txt
+++ b/backends/CMakeLists.txt
@@ -1,6 +1,11 @@
 add_subdirectory(fake)
 add_subdirectory(qscreen)
+
+if(${KF5Wayland_FOUND})
 add_subdirectory(kwayland)
+else()
+    message(STATUS "Not building kwayland backend, no KF5Wayland_FOUND set.")
+endif()
 
 if(${XCB_RANDR_FOUND})
     message(STATUS "Will build xrandr backend.")
-- 
2.5.0

