From af1cd7e415e047430278931e4af09766d3c8dc04 Mon Sep 17 00:00:00 2001
From: Helio Chissini de Castro <helio@kde.org>
Date: Sat, 20 Dec 2014 13:28:02 -0200
Subject: [PATCH] - Unify naming for a unique lxqt. No more suffixes

Upstream-Status: Applied

Signed-off-by: Helio Chissini de Castro <helio@kde.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index be8b01a..6bb3e57 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ set(CMAKE_AUTOMOC ON)
 find_package(Qt5Widgets REQUIRED QUIET)
 find_package(Qt5LinguistTools REQUIRED QUIET)
 
-find_package(lxqt-qt5 REQUIRED QUIET)
+find_package(lxqt REQUIRED QUIET)
 
 include(${LXQT_USE_FILE})
 
-- 
1.9.3

