From e770efcd3bcdbf0e885c8db2df10fa1bd09b4c06 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 17:56:45 +0200
Subject: [PATCH] fix build for QT_NO_SESSIONMANAGER
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/file/extractor/main.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/file/extractor/main.cpp b/src/file/extractor/main.cpp
index cf9fec7..7c12646 100644
--- a/src/file/extractor/main.cpp
+++ b/src/file/extractor/main.cpp
@@ -29,7 +29,9 @@
 #include <QByteArray>
 
 #include <QApplication>
+#ifndef QT_NO_SESSIONMANAGER
 #include <QSessionManager>
+#endif
 
 int main(int argc, char* argv[])
 {
@@ -47,11 +49,13 @@ int main(int argc, char* argv[])
 
     app.setQuitOnLastWindowClosed(false);
 
+#ifndef QT_NO_SESSIONMANAGER
     auto disableSessionManagement = [](QSessionManager &sm) {
         sm.setRestartHint(QSessionManager::RestartNever);
     };
     QObject::connect(&app, &QGuiApplication::commitDataRequest, disableSessionManagement);
     QObject::connect(&app, &QGuiApplication::saveStateRequest, disableSessionManagement);
+#endif
 
     Baloo::App appObject;
     return app.exec();
-- 
2.1.0

