From b20c48796484c2725be3020794f68a0bf8785455 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 15 Aug 2015 22:34:36 +0200
Subject: [PATCH] Baloo.pc.cmake: fix paths for Libs and Cflags
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/lib/Baloo.pc.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/lib/Baloo.pc.cmake b/src/lib/Baloo.pc.cmake
index 62eb504..05a79d6 100644
--- a/src/lib/Baloo.pc.cmake
+++ b/src/lib/Baloo.pc.cmake
@@ -8,5 +8,5 @@ Description: Baloo is a file indexing and searching framework for Linux
 URL: http://www.kde.org
 Requires: Qt5Core
 Version: ${BALOO_VERSION_STRING}
-Libs: -L${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR} -lKF5Baloo
-Cflags: -I${CMAKE_INSTALL_PREFIX}/${KF5_INCLUDE_INSTALL_DIR}/Baloo
+Libs: -L${LIB_INSTALL_DIR} -lKF5Baloo
+Cflags: -I${KF5_INCLUDE_INSTALL_DIR}/Baloo
-- 
2.1.0

