From 51a1a623dbdeac52b0afd383189e509f6b61df9e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 16 Sep 2015 15:20:30 +0200
Subject: [PATCH] align path of designer plugins
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 68b3d78..57c4792 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -199,7 +199,7 @@ add_feature_info("Qt Designer plugins" WITH_DESIGNER_PLUGIN "Marble widget suppo
 
 set(EXEC_INSTALL_PREFIX  ${CMAKE_INSTALL_PREFIX} CACHE PATH  "Base directory for executables and libraries" FORCE)
 if (QT5BUILD)
-   set(QT_PLUGINS_DIR ${CMAKE_INSTALL_LIBDIR}/plugins)
+   set(QT_PLUGINS_DIR ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/qt5/plugins)
 endif()
 
 ####################################################
-- 
2.1.0

