From 0689d84ea92dc35c69f97c4051844a4181ac2d19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 15 Aug 2016 23:42:41 +0200
Subject: [PATCH] compositorlauncher: work corrrectly with RaspberryPi VC4 drm
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 launcher/compositorlauncher.cpp | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/launcher/compositorlauncher.cpp b/launcher/compositorlauncher.cpp
index fd68c66..de9b86c 100644
--- a/launcher/compositorlauncher.cpp
+++ b/launcher/compositorlauncher.cpp
@@ -169,6 +169,15 @@ void CompositorLauncher::detectMode()
 
 void CompositorLauncher::detectHardware()
 {
+    // TODO: Detect Mali
+    // TODO: Detect Vivante
+
+    // Detect DRM
+    if (QDir(QStringLiteral("/sys/class/drm")).exists()) {
+        m_hardware = DrmHardware;
+        return;
+    }
+
     // Detect Broadcom
     bool found = deviceModel().startsWith(QStringLiteral("Raspberry"));
     if (!found) {
@@ -183,15 +192,6 @@ void CompositorLauncher::detectHardware()
         return;
     }
 
-    // TODO: Detect Mali
-    // TODO: Detect Vivante
-
-    // Detect DRM
-    if (QDir(QStringLiteral("/sys/class/drm")).exists()) {
-        m_hardware = DrmHardware;
-        return;
-    }
-
     // Unknown hardware
     m_hardware = UnknownHardware;
 }
-- 
2.5.5

