From 309df29de77eba24ac6ff37e792f16776fec3165 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 14 Jan 2014 13:25:07 +0100
Subject: [PATCH] GetGitRevision.cmake: call git directly - it is not found by
 cmake macros
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 cmake/GetGitRevision.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/GetGitRevision.cmake b/cmake/GetGitRevision.cmake
index 144b1ac..884eb3c 100644
--- a/cmake/GetGitRevision.cmake
+++ b/cmake/GetGitRevision.cmake
@@ -39,7 +39,7 @@ function(create_git_head_revision_file _file _target)
         add_custom_target(gitsha1
             ${CMAKE_COMMAND} -E remove -f ${CMAKE_CURRENT_BINARY_DIR}/${_file}
             COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/${_file}.in ${CMAKE_CURRENT_BINARY_DIR}/${_file}
-            COMMAND "${GIT_EXECUTABLE}" rev-parse HEAD >> ${CMAKE_CURRENT_BINARY_DIR}/${_file}
+            COMMAND git rev-parse HEAD >> ${CMAKE_CURRENT_BINARY_DIR}/${_file}
             WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
             VERBATIM)
         add_dependencies(${_target} gitsha1)
-- 
1.8.3.1

