From 5c5b3a0daa47009ede84061a904dbb43eceb61ed Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 6 Oct 2017 12:36:36 +0200
Subject: [PATCH] Make our cross build find parsetrigrams
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>

---
 data/CMakeLists.txt | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index 90077fa..b8876cc 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -14,17 +14,10 @@ TARGET_LINK_LIBRARIES(gentrigrams PUBLIC Qt5::Core KF5::SonnetCore)
 INSTALL(TARGETS parsetrigrams ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
 INSTALL(TARGETS gentrigrams ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
 
-if(CMAKE_CROSSCOMPILING AND PARSETRIGRAMS_EXECUTABLE)
-    add_executable(KF5::parsetrigrams IMPORTED GLOBAL)
-    set_target_properties(KF5::parsetrigrams PROPERTIES IMPORTED_LOCATION ${PARSETRIGRAMS_EXECUTABLE})
-else()
-    add_executable(KF5::parsetrigrams ALIAS parsetrigrams)
-endif()
-
 function(create_trigrams_map _target_name _in_DIR _out_FILE)
     add_custom_target(trigrams_${_target_name} ALL)
     add_custom_command(TARGET trigrams_${_target_name}
-                       COMMAND $<TARGET_FILE:KF5::parsetrigrams> "${_in_DIR}" > "${_out_FILE}"
+                       COMMAND parsetrigrams "${_in_DIR}" > "${_out_FILE}"
                       )
 
     add_dependencies(trigrams_${_target_name} parsetrigrams)
-- 
2.14.3

