From e08552046c304df50471e639780c8effdc5b593c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 9 Feb 2015 13:55:48 +0100
Subject: [PATCH] core: add valid extra plugin search path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/core/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
index 476bf22..a8ec795 100644
--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -14,7 +14,7 @@ set(sonnetcore_SRCS
 )
 
 # Dear packagers, this is just used as an extra search paths for plugins. Don't get your panties in a twist.
-add_definitions(-DINSTALLATION_PLUGIN_PATH="${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_PLUGINDIR}")
+add_definitions(-DINSTALLATION_PLUGIN_PATH="${KDE_INSTALL_PLUGINDIR}")
 
 
 add_library(KF5SonnetCore ${sonnetcore_SRCS})
-- 
1.9.3

