From 3fabfaa8f0beb09513bf0dd770f6273682edd12a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 15 Aug 2016 22:43:04 +0200
Subject: [PATCH] startplasmacompositor: align qt5 bin path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 startkde/startplasmacompositor.cmake | 16 ++++------------
 1 file changed, 4 insertions(+), 12 deletions(-)

diff --git a/startkde/startplasmacompositor.cmake b/startkde/startplasmacompositor.cmake
index 5bcf26a..6252d79 100644
--- a/startkde/startplasmacompositor.cmake
+++ b/startkde/startplasmacompositor.cmake
@@ -4,17 +4,9 @@
 #
 
 # in case we have been started with full pathname spec without being in PATH
-bindir=`echo "$0" | sed -n 's,^\(/.*\)/[^/][^/]*$,\1,p'`
-if [ -n "$bindir" ]; then
-  qbindir=`qtpaths --binaries-dir`
-  qdbus=$qbindir/qdbus
-  case $PATH in
-    $bindir|$bindir:*|*:$bindir|*:$bindir:*) ;;
-    *) PATH=$bindir:$PATH; export PATH;;
-  esac
-else
-  qdbus=qdbus
-fi
+
+# meta-qt5 specific
+qdbus=@OE_QMAKE_PATH_QT_BINS@/qdbus
 
 # We need to create config folder so we can write startupconfigkeys
 if [  ${XDG_CONFIG_HOME} ]; then
@@ -148,7 +140,7 @@ fi
 # better use the Autostart folder.
 
 # TODO: Use GenericConfigLocation once we depend on Qt 5.4
-scriptpath=`qtpaths --paths ConfigLocation | tr ':' '\n' | sed 's,$,/plasma-workspace,g'`
+scriptpath=`@OE_QMAKE_PATH_QT_BINS@/qtpaths --paths ConfigLocation | tr ':' '\n' | sed 's,$,/plasma-workspace,g'`
 
 # Add /env/ to the directory to locate the scripts to be sourced
 for prefix in `echo $scriptpath`; do
-- 
2.7.4

