From 3fabfaa8f0beb09513bf0dd770f6273682edd12a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 15 Aug 2016 22:43:04 +0200
Subject: [PATCH] startplasmacompositor: align qt5 bin path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>

---
 startkde/startplasmacompositor.cmake | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/startkde/startplasmacompositor.cmake b/startkde/startplasmacompositor.cmake
index 8ac47aa..55a016a 100644
--- a/startkde/startplasmacompositor.cmake
+++ b/startkde/startplasmacompositor.cmake
@@ -120,12 +120,12 @@ fi
 
 # Get a property value from org.freedesktop.locale1
 queryLocale1() {
-    qdbus --system org.freedesktop.locale1 /org/freedesktop/locale1 "$1"
+    @OE_QMAKE_PATH_QT_BINS@/qdbus --system org.freedesktop.locale1 /org/freedesktop/locale1 "$1"
 }
 
 # Query whether org.freedesktop.locale1 is available. If it is, try to
 # set XKB_DEFAULT_{MODEL,LAYOUT,VARIANT,OPTIONS} accordingly.
-if qdbus --system org.freedesktop.locale1 >/dev/null 2>/dev/null; then
+if @OE_QMAKE_PATH_QT_BINS@/qdbus --system org.freedesktop.locale1 >/dev/null 2>/dev/null; then
     # Do not overwrite existing values. There is no point in setting only some
     # of them as then they would not match anymore.
     if [ -z "${XKB_DEFAULT_MODEL}" -a -z "${XKB_DEFAULT_LAYOUT}" -a \
@@ -175,10 +175,10 @@ fi
 export XDG_DATA_DIRS
 
 # Make sure that D-Bus is running
-if qdbus >/dev/null 2>/dev/null; then
+if @OE_QMAKE_PATH_QT_BINS@/qdbus >/dev/null 2>/dev/null; then
     : # ok
 else
-    echo 'startplasmacompositor: Could not start D-Bus. Can you call qdbus?'  1>&2
+    echo 'startplasmacompositor: Could not start D-Bus. Can you call @OE_QMAKE_PATH_QT_BINS@/qdbus?'  1>&2
     test -n "$ksplash_pid" && kill "$ksplash_pid" 2>/dev/null
     exit 1
 fi
-- 
2.14.3

