From d5f3d3a0a02f01a78bb867929b854bb7df4861c3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 5 Sep 2018 00:07:23 +0200
Subject: [PATCH] startplasma.cmake: fix paths to qdbus
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>

---
 startkde/startplasma.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/startkde/startplasma.cmake b/startkde/startplasma.cmake
index f7330ab3..4fddd097 100644
--- a/startkde/startplasma.cmake
+++ b/startkde/startplasma.cmake
@@ -169,13 +169,13 @@ if test x"$wait_drkonqi"x = x"true"x ; then
     # wait for remaining drkonqi instances with timeout (in seconds)
     wait_drkonqi_timeout=`kreadconfig5 --file startkderc --group WaitForDrKonqi --key Timeout --default 900`
     wait_drkonqi_counter=0
-    while qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
+    while @OE_QMAKE_PATH_QT_BINS@/qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
         sleep 5
         wait_drkonqi_counter=$((wait_drkonqi_counter+5))
         if test "$wait_drkonqi_counter" -ge "$wait_drkonqi_timeout" ; then
             # ask remaining drkonqis to die in a graceful way
-            qdbus | grep 'org.kde.drkonqi-' | while read address ; do
-                qdbus "$address" "/MainApplication" "quit"
+            @OE_QMAKE_PATH_QT_BINS@/qdbus | grep 'org.kde.drkonqi-' | while read address ; do
+                @OE_QMAKE_PATH_QT_BINS@/qdbus "$address" "/MainApplication" "quit"
             done
             break
         fi
-- 
2.14.4

