From cfecefeea4d3d895442466c10ce4813a4ef548c7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 2 Aug 2017 15:47:30 +0200
Subject: [PATCH] Don't disable EXPERIMENTAL_PLUGINS
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 source/Makefile.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/Makefile.mk b/source/Makefile.mk
index 5db2b76..fd320e1 100644
--- a/source/Makefile.mk
+++ b/source/Makefile.mk
@@ -11,7 +11,7 @@
 CARLA_VESTIGE_HEADER = true
 
 # Enable experimental plugins, don't complain if the build fails when using this!
-EXPERIMENTAL_PLUGINS = false
+#EXPERIMENTAL_PLUGINS = false
 
 # --------------------------------------------------------------
 # DO NOT MODIFY PAST THIS POINT!
-- 
2.9.4

