From c25739c331979d96b209aee3f97a35c99908679c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 2 Aug 2017 01:29:39 +0200
Subject: [PATCH] native-plugins: don't kill LDFLAGS set by build system
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 source/native-plugins/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/source/native-plugins/Makefile b/source/native-plugins/Makefile
index f28acd1..fd6347c 100644
--- a/source/native-plugins/Makefile
+++ b/source/native-plugins/Makefile
@@ -207,22 +207,22 @@ $(MODULEDIR)/$(MODULENAME).a: $(OBJS)
 resources/at1-ui$(APP_EXT): $(OBJDIR)/zita-at1-ui.cpp.o
 	-@mkdir -p $(OBJDIR)
 	@echo "Linking at1-ui"
-	@$(CXX) $^ $(ZITA_UI_LINK_FLAGS) -o $@
+	@$(CXX) $^ $(LDFLAGS) $(ZITA_UI_LINK_FLAGS) -o $@
 
 resources/bls1-ui$(APP_EXT): $(OBJDIR)/zita-bls1-ui.cpp.o
 	-@mkdir -p $(OBJDIR)
 	@echo "Linking bls1-ui"
-	@$(CXX) $^ $(ZITA_UI_LINK_FLAGS) -o $@
+	@$(CXX) $^ $(LDFLAGS) $(ZITA_UI_LINK_FLAGS) -o $@
 
 resources/rev1-ui$(APP_EXT): $(OBJDIR)/zita-rev1-ui.cpp.o
 	-@mkdir -p $(OBJDIR)
 	@echo "Linking rev1-ui"
-	@$(CXX) $^ $(ZITA_UI_LINK_FLAGS) -o $@
+	@$(CXX) $^ $(LDFLAGS) $(ZITA_UI_LINK_FLAGS) -o $@
 
 resources/zynaddsubfx-ui$(APP_EXT): $(OBJDIR)/zynaddsubfx-ui.cpp.o
 	-@mkdir -p $(OBJDIR)
 	@echo "Linking zynaddsubfx-ui"
-	@$(CXX) $^ $(ZYN_LD_FLAGS) -o $@
+	@$(CXX) $^ $(LDFLAGS) $(ZYN_LD_FLAGS) -o $@
 
 # ----------------------------------------------------------------------------------------------------------------------------
 
-- 
2.9.4

