From 875457d88e933109393fc24b41338d7f301d771d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 3 Aug 2017 12:16:16 +0200
Subject: [PATCH] do not autodetect pyqt - it won't work in oe
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 source/Makefile.mk | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/source/Makefile.mk b/source/Makefile.mk
index 5db2b76..1190f25 100644
--- a/source/Makefile.mk
+++ b/source/Makefile.mk
@@ -256,26 +256,6 @@ HAVE_QT=true
 endif
 
 # --------------------------------------------------------------
-# Set PyQt tools
-
-PYUIC4 ?= /usr/bin/pyuic4
-PYUIC5 ?= /usr/bin/pyuic5
-
-ifneq (,$(wildcard $(PYUIC4)))
-HAVE_PYQT=true
-HAVE_PYQT4=true
-else
-HAVE_PYQT4=false
-endif
-
-ifneq (,$(wildcard $(PYUIC5)))
-HAVE_PYQT=true
-HAVE_PYQT5=true
-else
-HAVE_PYQT5=false
-endif
-
-# --------------------------------------------------------------
 # Set default Qt used in frontend
 
 ifeq ($(HAVE_PYQT4),true)
-- 
2.9.4

