From 1f27d30ce71ac606be3010d36a70fa12964f1cb4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 5 Sep 2017 18:12:47 +0200
Subject: [PATCH] do not build zynaddsubfx - we have a better version
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 source/Makefile.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/Makefile.mk b/source/Makefile.mk
index b230f31..fa30528 100644
--- a/source/Makefile.mk
+++ b/source/Makefile.mk
@@ -407,7 +407,7 @@ endif
 # --------------------------------------------------------------
 # Set libs stuff (part 3)
 
-HAVE_ZYN_DEPS    = $(shell pkg-config --exists liblo fftw3 mxml zlib && echo true)
+HAVE_ZYN_DEPS    = false
 ifeq ($(HAVE_FLTK),true)
 HAVE_ZYN_UI_DEPS = true
 endif
-- 
2.9.5

