From 6ec28a23f2850f6e06e3637b8af2bb956f5a448b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 18:21:50 +0200
Subject: [PATCH] fix build with QT_NO_SESSIONMANAGER set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>

---
 src/manager/kwalletmanager.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/manager/kwalletmanager.cpp b/src/manager/kwalletmanager.cpp
index 2672c7c..c505d23 100644
--- a/src/manager/kwalletmanager.cpp
+++ b/src/manager/kwalletmanager.cpp
@@ -100,12 +100,14 @@ void KWalletManager::configUI() {
                 break;
             }
         }
+#ifndef QT_NO_SESSIONMANAGER
         if (!isOpen && qApp->isSessionRestored()) {
             delete _tray;
             _tray = nullptr;
             QTimer::singleShot(0, qApp, SLOT(quit()));
             return;
         }
+#endif
     } else {
         _tray = nullptr;
     }
-- 
2.1.0

