From d1c609b320eadbe7089b0482353ffef576813ca2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luis=20D=C3=ADaz=20M=C3=A1s?= <piponazo@gmail.com>
Date: Tue, 18 Dec 2018 08:05:16 +0100
Subject: [PATCH 22/70] cmake: man pages only installed with the exiv2 app

---
 CMakeLists.txt     | 3 ---
 src/CMakeLists.txt | 7 +------
 2 files changed, 1 insertion(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8dfd9d97..e98cfac1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -106,6 +106,3 @@ configure_file(cmake/exiv2.pc.in exiv2.pc @ONLY)
 install(FILES ${CMAKE_BINARY_DIR}/exiv2.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 
 include(cmake/printSummary.cmake)
-
-# That's all Folks!
-##
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ca24006e..0d093924 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -273,11 +273,6 @@ if(EXIV2_BUILD_EXIV2_COMMAND)
     endif()
 
     install(TARGETS exiv2 RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+    install( FILES exiv2.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 )
 endif()
 
-# ******************************************************************************
-# Man page
-install( FILES exiv2.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 )
-
-# That's all Folks!
-##
-- 
2.17.2

