From 2a859d65af1fdc45495aa0e2b9ab562a68618a1d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 24 Nov 2014 11:26:08 +0100
Subject: [PATCH] install server library
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/server/CMakeLists.txt | 42 +++++++++++++++++++++---------------------
 1 file changed, 21 insertions(+), 21 deletions(-)

diff --git a/src/server/CMakeLists.txt b/src/server/CMakeLists.txt
index bd64770..43c47d9 100644
--- a/src/server/CMakeLists.txt
+++ b/src/server/CMakeLists.txt
@@ -46,24 +46,24 @@ set_target_properties(KF5WaylandServer PROPERTIES VERSION   ${KWAYLAND_VERSION_S
                                                  EXPORT_NAME WaylandServer
 )
 
-# install(TARGETS KF5WaylandServer EXPORT KF5WaylandTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
-#
-# install(FILES
-#   ${CMAKE_CURRENT_BINARY_DIR}/KWayland/Server/kwaylandserver_export.h
-#   buffer_interface.h
-#   clientconnection.h
-#   compositor_interface.h
-#   datadevice_interface.h
-#   datadevicemanager_interface.h
-#   dataoffer_interface.h
-#   datasource_interface.h
-#   display.h
-#   global.h
-#   output_interface.h
-#   region_interface.h
-#   resource.h
-#   seat_interface.h
-#   shell_interface.h
-#   surface_interface.h
-#   DESTINATION ${KF5_INCLUDE_INSTALL_DIR}/KWayland/Server COMPONENT Devel
-# )
+install(TARGETS KF5WaylandServer EXPORT KF5WaylandTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
+
+install(FILES
+  ${CMAKE_CURRENT_BINARY_DIR}/KWayland/Server/kwaylandserver_export.h
+  buffer_interface.h
+  clientconnection.h
+  compositor_interface.h
+  datadevice_interface.h
+  datadevicemanager_interface.h
+  dataoffer_interface.h
+  datasource_interface.h
+  display.h
+  global.h
+  output_interface.h
+  region_interface.h
+  resource.h
+  seat_interface.h
+  shell_interface.h
+  surface_interface.h
+  DESTINATION ${KF5_INCLUDE_INSTALL_DIR}/KWayland/Server COMPONENT Devel
+)
-- 
1.8.3.1

