From b8673fb304f0eed41a1bcaa953a63bedb27b05e6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 19 Sep 2016 01:11:47 +0200
Subject: [PATCH] do not check for known machine types
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 build/mixxx.py | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/build/mixxx.py b/build/mixxx.py
index aeed9e8..631bbfa 100644
--- a/build/mixxx.py
+++ b/build/mixxx.py
@@ -45,20 +45,6 @@ class MixxxBuild(object):
         if target not in ['windows', 'osx', 'linux', 'bsd']:
             raise Exception("invalid target platform")
 
-        if machine.lower() not in ['x86_64', 'x86', 'i686', 'i586',
-                                   'alpha', 'hppa', 'mips', 'mipsel', 's390',
-                                   'sparc', 'ia64', 'armel', 'armhf', 'hurd-i386',
-                                   'armv5tel', 'armv5tejl', 'armv6l', 'armv6hl',
-                                   'armv7l', 'armv7hl', 'armv7hnl',
-                                   'sh3', 'sh4',
-                                   'kfreebsd-amd64', 'kfreebsd-i386',
-                                   'i486', 'i386', 'ppc', 'ppc64', 'powerpc',
-                                   'powerpc64', 'powerpcspe', 's390x',
-                                   'amd64', 'em64t', 'intel64', 'arm64',
-                                   'ppc64el', 'ppc64le', 'm68k', 'mips64',
-                                   'mips64el', 'mipsn32', 'mipsn32el',
-                                   'aarch64']:
-            raise Exception("invalid machine type")
 
         if toolchain not in ['gnu', 'msvs']:
             raise Exception('invalid toolchain type')
-- 
2.9.4

