From a16c3cf78bdfc954560812478e0022957acaca8d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 18 Nov 2018 23:16:19 +0100
Subject: [PATCH] Do not add QT libdir - it injectst /usr/lib these days
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 build/depends.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/build/depends.py b/build/depends.py
index e6c4a4d250..637ebd0bb9 100644
--- a/build/depends.py
+++ b/build/depends.py
@@ -484,11 +484,6 @@ class Qt(Dependence):
         if build.platform_is_osx:
             compiling_on_104 = (
                 os.popen('sw_vers').readlines()[1].find('10.4') >= 0)
-        if not build.platform_is_windows and not (using_104_sdk or compiling_on_104):
-            qtdir = build.env['QTDIR']
-            framework_path = Qt.find_framework_libdir(qtdir, qt5)
-            if os.path.isdir(framework_path):
-                build.env.Append(LINKFLAGS="-L" + framework_path)
 
         # Mixxx requires C++11 support. Windows enables C++11 features by
         # default but Clang/GCC require a flag.
-- 
2.14.4

