From c2fca3345cfbcb0fc129ea23e8438057202604f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 4 Aug 2017 22:01:03 +0200
Subject: [PATCH] do not perform link test durin cross build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/ladspa/wscript | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/ladspa/wscript b/src/ladspa/wscript
index be5f31d..2c43e2d 100644
--- a/src/ladspa/wscript
+++ b/src/ladspa/wscript
@@ -10,10 +10,6 @@ def configure(conf):
 def test_loadable(self):
     def link_and_test(lib=self.link_task.outputs[0].bldpath(self.env), link=self.link_task.run):
         link()
-        try:
-            ctypes.cdll.LoadLibrary(lib)
-        except Exception as e:
-            raise RuntimeError("LADSPA load test failed: %s" % e)
     self.link_task.run = link_and_test
 
 def build(bld):
-- 
2.9.4

