From 632015125044fad5b268329570b55246002cd792 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 13 Jun 2021 23:11:35 +0200
Subject: [PATCH] Do not build po-files on native build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ba41271..11ae64d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,6 +68,7 @@ include(KF5PackageMacros.cmake)
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 ################# list the subdirectories #################
+if(false)
 if (KF5DocTools_FOUND)
     add_subdirectory(docs)
 endif()
@@ -76,6 +77,7 @@ ki18n_install(po)
 if (KF5DocTools_FOUND)
     kdoctools_install(po)
 endif()
+endif()
 add_subdirectory(src)
 if (BUILD_TESTING)
     add_subdirectory(autotests)
-- 
2.31.1

