From 938ed57e8985770b1e9824a1d1de3d67b3b2f2f7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 25 Aug 2016 11:30:33 +0200
Subject: [PATCH] do build kpackagetool only on native
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7da9a1d..b701045 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,6 +74,7 @@ include(KF5PackageMacros.cmake)
 # make package_version.h available
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
+if (false)
 ################# list the subdirectories #################
 if (KF5DocTools_FOUND)
     add_subdirectory(docs)
@@ -85,6 +86,7 @@ if (IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
         kdoctools_install(po)
     endif()
 endif()
+endif()
 
 add_subdirectory(src)
 if (BUILD_TESTING)
-- 
2.14.4

