From 508b26cd817083058aeaadb2d173e93545c5a938 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 16 Dec 2014 22:40:40 +0100
Subject: [PATCH 10/12] align phonon path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 phonon/platform_kde/kiomediastream.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/phonon/platform_kde/kiomediastream.h b/phonon/platform_kde/kiomediastream.h
index ebc2700..1d17727 100644
--- a/phonon/platform_kde/kiomediastream.h
+++ b/phonon/platform_kde/kiomediastream.h
@@ -20,7 +20,7 @@
 #ifndef PHONON_KIOMEDIASTREAM_H
 #define PHONON_KIOMEDIASTREAM_H
 
-#include <Phonon/AbstractMediaStream>
+#include <phonon/AbstractMediaStream>
 #include <KIO/Job>
 
 class QUrl;
-- 
1.8.3.1

