From 2001810ae3b816e9bf094ab31678d21247ce8045 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 18 Dec 2014 17:21:18 +0100
Subject: [PATCH 12/12] Make KWayland optional
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d86e815..08bb296 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,7 @@ if(X11_FOUND AND XCB_XCB_FOUND)
   set(HAVE_X11 1)
 endif()
 
-find_package(KF5Wayland CONFIG REQUIRED COMPONENTS WaylandClient)
+find_package(KF5Wayland CONFIG COMPONENTS WaylandClient)
 set_package_properties(KF5Wayland PROPERTIES
     TYPE OPTIONAL
     PURPOSE "Required for the workspace on Wayland")
-- 
1.8.3.1

