From 12976fcbcc5b68e7e9e7c6deba3983a7355f8f2f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 25 Jul 2015 22:29:29 +0200
Subject: [PATCH] slightly change theme defaults
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 config/lxqt.conf | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/config/lxqt.conf b/config/lxqt.conf
index 1d48e56..5018618 100644
--- a/config/lxqt.conf
+++ b/config/lxqt.conf
@@ -7,5 +7,8 @@ tool_button_style=ToolButtonTextBesideIcon
 [Qt]
 font="Sans,11,-1,5,50,0,0,0,0,0"
 doubleClickInterval=400
-style=Fusion
+style=Oxygen
 wheelScrollLines=3
+
+[Mouse]
+cursor_theme=Oxygen_Zion
-- 
1.9.3

