From 424d64bdf12632bfd7d3331a4aef0f2bfdf7de8d Mon Sep 17 00:00:00 2001
From: "Luis Gustavo S. Barreto" <gustavo@ossystems.com.br>
Date: Thu, 12 Nov 2015 17:32:17 +0000
Subject: [PATCH] Ensure xkb_base is not empty instead of checking if a file
 exists

Upstream-Status: Pending
---
 ConfigureChecks.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ConfigureChecks.cmake b/ConfigureChecks.cmake
index f8272c4..e8522d3 100644
--- a/ConfigureChecks.cmake
+++ b/ConfigureChecks.cmake
@@ -2,10 +2,10 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_P
 
 execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=xkb_base xkeyboard-config OUTPUT_VARIABLE XKBDIR OUTPUT_STRIP_TRAILING_WHITESPACE)
 if (CMAKE_CROSSCOMPILING)
-    if (NOT EXISTS "${CMAKE_SYSROOT}/${XKBDIR}")
+    if ("${XKBDIR}" STREQUAL "")
       message(FATAL_ERROR "Couldn't find XKB location in CMAKE_SYSROOT: \"${CMAKE_SYSROOT}/${XKBDIR}\"")
     endif()
-elseif(NOT EXISTS "${XKBDIR}")
+elseif ("${XKBDIR}" STREQUAL "")
     message(FATAL_ERROR "Couldn't find XKB location: \"${XKBDIR}\".")
 endif()
 set(KWIN_BIN "kwin_x11" CACHE STRING "Name of the KWin binary")
-- 
2.31.1

