From b7285c71f5b04bc55441217bf963078b38d6bad4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 21:41:27 +0200
Subject: [PATCH] fix build with QT_NO_SESSIONMANAGER set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>

---
 app/main.cpp | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/app/main.cpp b/app/main.cpp
index 0396400..5924683 100644
--- a/app/main.cpp
+++ b/app/main.cpp
@@ -206,6 +206,7 @@ int main(int argc, char **argv)
     // This is needed to prevent Dolphin from freezing when opening an archive.
     KDBusService dbusService(KDBusService::Multiple | KDBusService::NoExitOnFailure);
 
+#ifndef QT_NO_SESSIONMANAGER
     // Session restoring.
     if (application.isSessionRestored()) {
         if (!KMainWindow::canBeRestored(1)) {
@@ -218,8 +219,9 @@ int main(int argc, char **argv)
             delete window;
             return -1;
         }
-    } else { // New ark window (no restored session).
-
+    } else // New ark window (no restored session).
+#endif
+    {
         // Open any given URLs.
         const QStringList urls = parser.positionalArguments();
 
-- 
2.5.5

