From b7285c71f5b04bc55441217bf963078b38d6bad4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 21:41:27 +0200
Subject: [PATCH] fix build with QT_NO_SESSIONMANAGER set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 app/main.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/app/main.cpp b/app/main.cpp
index 4a95063..c3d3a61 100644
--- a/app/main.cpp
+++ b/app/main.cpp
@@ -175,6 +175,7 @@ int main(int argc, char **argv)
     KDBusService dbusService(KDBusService::Multiple);
 
     //session restoring
+#ifndef QT_NO_SESSIONMANAGER
     if (application.isSessionRestored()) {
         if (!KMainWindow::canBeRestored(1)) {
             return -1;
@@ -186,7 +187,9 @@ int main(int argc, char **argv)
             delete window;
             return -1;
         }
-    } else { //new ark window (no restored session)
+    } else
+#endif
+    { //new ark window (no restored session)
 
         // open any given URLs
         const QStringList urls = parser.positionalArguments();
-- 
2.1.0

