From 530a31e3eb24b2f1ed7b30859ed62a2d1b24bd91 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 7 Feb 2017 11:51:58 +0100
Subject: [PATCH] add a configure option to disable build of man-pages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure.ac | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8316c67..099fc0c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -67,6 +67,8 @@ AC_ARG_ENABLE([largefile],
     [AS_HELP_STRING([--disable-largefile],[disable LFS (large file support)])])
 AC_ARG_ENABLE([util],
     [AS_HELP_STRING([--disable-util],[disable build of command-line utilities])])
+AC_ARG_ENABLE([man],
+    [AS_HELP_STRING([--disable-man],[disable build of man-pages])])
 AC_ARG_ENABLE([bi],
     [AS_HELP_STRING([--enable-bi=ARCH],[enable -mARCH for bi-arch compilation])])
 AC_ARG_ENABLE([ub],
@@ -395,7 +397,7 @@ AM_CONDITIONAL([ADD_PLATFORM_POSIX],[test "$X_PLATFORM" = "posix"])
 AM_CONDITIONAL([ADD_PLATFORM_WIN32],[test "$X_PLATFORM" = "win32"])
 
 AM_CONDITIONAL([ADD_UTIL],[test "$enable_util" != "no"])
-AM_CONDITIONAL([ADD_MANS],[test "$X_PLATFORM" != "win32"])
+AM_CONDITIONAL([ADD_MANS],[test "$enable_man" != "no"])
 
 ###############################################################################
 # declare common substitutions
-- 
2.9.3

