From bb8ed404d0a5a7bc4ed5642e07e4440a5cb278a8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 9 Feb 2015 12:20:27 +0100
Subject: [PATCH] kdeinit: fix paths to libraries
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/kdeinit/kinit.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/kdeinit/kinit.cpp b/src/kdeinit/kinit.cpp
index 3c3c913..cd1dc9c 100644
--- a/src/kdeinit/kinit.cpp
+++ b/src/kdeinit/kinit.cpp
@@ -657,7 +657,7 @@ static pid_t launch(int argc, const char *_name, const char *args,
                     //     https://sourceware.org/bugzilla/show_bug.cgi?id=13945
                     //     - so we try hacking it in ourselves
                     QString install_lib_dir = QFile::decodeName(
-                            CMAKE_INSTALL_PREFIX "/" LIB_INSTALL_DIR "/");
+                            LIB_INSTALL_DIR "/");
                     libpath = install_lib_dir + libpath;
                     l.setFileName(libpath);
                     l.load();
@@ -1555,7 +1555,7 @@ static int initXconnection()
 // Completely unrelated to plugins.
 static QString findSharedLib(const QString &lib)
 {
-    QString path = QFile::decodeName(CMAKE_INSTALL_PREFIX "/" LIB_INSTALL_DIR "/") + lib;
+    QString path = QFile::decodeName(LIB_INSTALL_DIR "/") + lib;
     if (QFile::exists(path)) {
         return path;
     }
-- 
1.9.3

