From 43251e9b177f5256825deaffe136230ca3c0378a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 18 Sep 2016 22:14:59 +0200
Subject: [PATCH 1/2] do not pin build flags - use defaults
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross-specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
Signed-off-by: wata2ki <wata2ki@gmail.com>
---
 src/makefile | 16 +++++-----------
 1 file changed, 5 insertions(+), 11 deletions(-)

diff --git a/src/makefile b/src/makefile
index 886237f..1764da7 100644
--- a/src/makefile
+++ b/src/makefile
@@ -15,8 +15,6 @@ INSTALL_BINARY_DIR	=	/usr/bin/
 
 INCLUDES	=	-I.
 LIBRARIES	=	-ldl -lm
-CFLAGS		=	$(INCLUDES) -Wall -Werror -O3 -fPIC
-CXXFLAGS	=	$(CFLAGS)
 PLUGINS		=	../plugins/amp.so				\
 			../plugins/delay.so				\
 			../plugins/filter.so				\
@@ -25,8 +23,6 @@ PLUGINS		=	../plugins/amp.so				\
 PROGRAMS	=	../bin/analyseplugin				\
 			../bin/applyplugin 				\
 			../bin/listplugins
-CC		=	cc
-CPP		=	c++
 
 ###############################################################################
 #
@@ -34,12 +30,10 @@ CPP		=	c++
 #
 
 ../plugins/%.so:	plugins/%.c ladspa.h
-	$(CC) $(CFLAGS) -o plugins/$*.o -c plugins/$*.c
-	$(LD) -o ../plugins/$*.so plugins/$*.o -shared
+	$(CC) $(CFLAGS) -o ../plugins/$*.so plugins/$*.c -shared $(LDFLAGS)
 
 ../plugins/%.so:	plugins/%.cpp ladspa.h
-	$(CPP) $(CXXFLAGS) -o plugins/$*.o -c plugins/$*.cpp
-	$(CPP) -o ../plugins/$*.so plugins/$*.o -shared
+	$(CXX) $(CFLAGS) -o ../plugins/$*.so plugins/$*.cpp -shared $(LDFLAGS)
 
 ###############################################################################
 #
@@ -90,17 +84,17 @@ targets:	$(PLUGINS) $(PROGRAMS)
 #
 
 ../bin/applyplugin:	applyplugin.o load.o default.o
-	$(CC) $(CFLAGS) $(LIBRARIES)					\
+	$(CC) $(CFLAGS) -ldl $(LDFLAGS)					\
 		-o ../bin/applyplugin					\
 		applyplugin.o load.o default.o
 
 ../bin/analyseplugin:	analyseplugin.o load.o default.o
-	$(CC) $(CFLAGS) $(LIBRARIES)					\
+	$(CC) $(CFLAGS) -ldl -lm $(LDFLAGS)					\
 		-o ../bin/analyseplugin 				\
 		analyseplugin.o load.o default.o
 
 ../bin/listplugins:	listplugins.o search.o
-	$(CC) $(CFLAGS) $(LIBRARIES)					\
+	$(CC) $(CFLAGS) -ldl $(LDFLAGS)					\
 		-o ../bin/listplugins	 				\
 		listplugins.o search.o
 
-- 
2.7.4

