From d40bc302b7d5f7c70fb9eaf25f79e735a71dd786 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 6 Nov 2022 16:17:47 +0100
Subject: [PATCH] config/CMakeLists.txt: Use relative path to include
 meta_types.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 config/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/CMakeLists.txt b/config/CMakeLists.txt
index 65d291b..f618645 100644
--- a/config/CMakeLists.txt
+++ b/config/CMakeLists.txt
@@ -38,7 +38,7 @@ set(${PROJECT_NAME}_DBUS_INTERFACES
 )
 set_source_files_properties(${${PROJECT_NAME}_PATH_TO_DAEMON}/org.lxqt.global_key_shortcuts.daemon.xml PROPERTIES
     BASENAME org.lxqt.global_key_shortcuts.daemon
-    INCLUDE ${${PROJECT_NAME}_PATH_TO_DAEMON}/meta_types.h
+    INCLUDE ../daemon/meta_types.h
 )
 
 set(${PROJECT_NAME}_HEADERS
-- 
2.38.1

