From 6576ed9befb6c4d53dbb893068145b1daef82675 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 12 Feb 2022 22:57:07 +0100
Subject: [PATCH] Remove kwin-strip-effect-metadata find/build dance
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 src/effects/CMakeLists.txt | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/src/effects/CMakeLists.txt b/src/effects/CMakeLists.txt
index 9d06969..257f086 100644
--- a/src/effects/CMakeLists.txt
+++ b/src/effects/CMakeLists.txt
@@ -8,22 +8,6 @@ function(kwin_add_effect_config name)
     kcoreaddons_add_plugin(${name} INSTALL_NAMESPACE "kwin/effects/configs" SOURCES ${ARGV})
 endfunction()
 
-if (CMAKE_CROSSCOMPILING)
-    if (NOT KWIN_HOST_TOOLING)
-        message(FATAL_ERROR "Please provide the path to a native build and pass -DKWIN_HOST_TOOLING=path")
-    endif()
-    find_file(KSEM_EXECUTABLE StripEffectMetadataTargets.cmake
-              PATHS ${KWIN_HOST_TOOLING}
-              NO_DEFAULT_PATH
-              NO_CMAKE_FIND_ROOT_PATH)
-    include(${KSEM_EXECUTABLE})
-else()
-    add_executable(kwin-strip-effect-metadata strip-effect-metadata.cpp)
-    target_link_libraries(kwin-strip-effect-metadata Qt5::Core)
-    # Export the kwin-strip-effect-metadata target to a file, so it can be used by another build.
-    export(TARGETS kwin-strip-effect-metadata FILE ${CMAKE_BINARY_DIR}/StripEffectMetadataTargets.cmake)
-endif()
-
 function (kwin_strip_builtin_effect_metadata target metadata)
     set(stripped_metadata "${CMAKE_CURRENT_BINARY_DIR}/${metadata}.stripped")
 
-- 
2.34.1

