From d372a1593e1d22e84034ac3ea14d401d5ae4ef71 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 17 Jun 2015 10:00:25 +0200
Subject: [PATCH 1/2] kconfiggui: do only compile in case sessions are
 avaliable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/gui/kconfiggui.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gui/kconfiggui.cpp b/src/gui/kconfiggui.cpp
index 67b6009..e313ff4 100644
--- a/src/gui/kconfiggui.cpp
+++ b/src/gui/kconfiggui.cpp
@@ -25,6 +25,7 @@
 
 #include <kconfig.h>
 
+#ifndef QT_NO_SESSIONMANAGER
 static QString configName(const QString &id, const QString &key)
 {
     return(QLatin1String("session/") + QGuiApplication::applicationName() +
@@ -73,3 +74,4 @@ QString KConfigGui::sessionConfigName()
     return sessionConfig()->name();
 }
 #endif
+#endif // QT_NO_SESSIONMANAGER
-- 
1.9.3
