From 19c7b3eaa79eb7d6ed9037d6c9efac422356889d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 23 Jul 2015 16:34:45 +0200
Subject: [PATCH] finding sed does not work and is not neccessary
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt     | 2 --
 src/CMakeLists.txt | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9ed9cf4..5a611d2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,8 +2,6 @@ project(obconf-qt)
 # newer cmake is required for CMAKE_AUTOMOC Qt support
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
-find_program(SED_PROGRAM sed)
-
 include(GNUInstallDirs)
 
 set(LXQTBT_MINIMUM_VERSION "0.5.0")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 88196b4..fb2d6bb 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -48,7 +48,7 @@ set_property(SOURCE ${obconf-qt_UI_H} PROPERTY SKIP_AUTOMOC ON)
 # but we don't want that. Fix it with sed.
 add_custom_command(
   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/ui_obconf_fixed.h
-  COMMAND ${SED_PROGRAM}
+  COMMAND sed
   ARGS "/connectSlotsByName/d" ${obconf-qt_UI_H} > ${CMAKE_CURRENT_BINARY_DIR}/ui_obconf_fixed.h
   # DEPENDS
   MAIN_DEPENDENCY ${obconf-qt_UI_H}
-- 
2.13.6

