From d7cf023376cb834f9becb63f2648c63c5ec3fd12 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 13 Dec 2020 14:08:12 +0100
Subject: [PATCH] CMakeLists.txt: make MLT_PREFIX configurable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b25f04..fff0f52 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,7 @@ if(WIN32)
     find_package(DrMinGW)
     set(MLT_PREFIX "..")
 else()
-    set(MLT_PREFIX ${MLT_ROOT_DIR})
+    set(MLT_PREFIX ${MLT_ROOT_DIR} CACHE STRING "mlt prefix")
     find_package(Qt5 OPTIONAL_COMPONENTS WebEngineWidgets QUIET)
 endif()
 
-- 
2.26.2

