From d7cf023376cb834f9becb63f2648c63c5ec3fd12 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Thu, 15 Nov 2018 18:46:38 +0100
Subject: [PATCH] CMakeLists.txt: make MLT_PREFIX configurable

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec69e0c..0792bc3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -119,7 +119,7 @@ set_package_properties(MLT PROPERTIES
                 PURPOSE "Required to do video processing")
 
 if(NOT WIN32)
-    set(MLT_PREFIX ${MLT_ROOT_DIR})
+    set(MLT_PREFIX ${MLT_ROOT_DIR} CACHE STRING "mlt prefix")
 else()
     set(MLT_PREFIX "..")
 endif()
-- 
2.14.5

