From a9c1fc97b0b8e0586b799955e8909937e88e81c1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 8 Apr 2016 00:45:05 +0200
Subject: [PATCH] hardwareintegration: glx compositor does not build in GLES
 environments
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/hardwareintegration/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hardwareintegration/CMakeLists.txt b/src/hardwareintegration/CMakeLists.txt
index a10419c..6de68e8 100644
--- a/src/hardwareintegration/CMakeLists.txt
+++ b/src/hardwareintegration/CMakeLists.txt
@@ -9,7 +9,7 @@ if(X11_FOUND AND X11_Xcomposite_FOUND)
     if(EGL_FOUND)
         add_subdirectory(xcomposite-egl)
     endif()
-    if(OPENGL_FOUND)
+    if(${Qt5Gui_OPENGL_IMPLEMENTATION} STREQUAL "GL")
         add_subdirectory(xcomposite-glx)
     endif()
 endif()
-- 
2.5.5

