From 616528cad6a98b2604eb8365f55538a738e7db2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 21:34:06 +0200
Subject: [PATCH] fix build with QT_NO_SESSIONMANAGER set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 main.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 311268d..e0257f1 100644
--- a/main.cpp
+++ b/main.cpp
@@ -84,9 +84,12 @@ int main(int argc, char **argv)
 
     KDBusService service;
 
+#ifndef QT_NO_SESSIONMANAGER
     if (app.isSessionRestored())
         RESTORE(KBounceMainWindow)
-    else {
+    else
+#endif
+    {
         KBounceMainWindow *w = new KBounceMainWindow;
         w->show();
     }
-- 
2.5.5

