From 20628c130547773e618aa83ddda156147113a24b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 9 Feb 2015 14:10:24 +0100
Subject: [PATCH] fix path to ksendbugmail
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/kbugreport.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kbugreport.cpp b/src/kbugreport.cpp
index c275129..41cf0d4 100644
--- a/src/kbugreport.cpp
+++ b/src/kbugreport.cpp
@@ -528,7 +528,7 @@ bool KBugReport::sendBugReport()
 
     QString command = QStandardPaths::findExecutable(QStringLiteral("ksendbugmail"));
     if (command.isEmpty()) {
-        command = QFile::decodeName(CMAKE_INSTALL_PREFIX "/" KF5_LIBEXEC_INSTALL_DIR "/ksendbugmail");
+        command = QFile::decodeName(KF5_LIBEXEC_INSTALL_DIR "/ksendbugmail");
     }
 
     QProcess proc;
-- 
1.9.3

