From ff9690f643d36aefe01494c3f4ae639888448f2a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 18 Sep 2016 20:05:35 +0200
Subject: [PATCH] do not perform test - it tries to run cross binaries
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross-specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 7cba23b..a84120d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -25,7 +25,7 @@ if test "x$PROGS_ARGUMENT" = "xyes" ; then
 	PKG_CHECK_MODULES([SNDFILE],[sndfile],have_sndfile="yes",have_sndfile="no")
 	if test "x$have_sndfile" = "xyes" ; then
 		dnl all good, build everything
-		TARGETS="sdk plugins host rdfgen test"
+		TARGETS="sdk plugins host rdfgen"
 	else
 		dnl no libsndfile - can't build host!
 		AC_MSG_ERROR([libsndfile not found - cannot build example Vamp host!
-- 
2.5.5

