From 87a5d75ec4e9a804850bd8e2b3a69e364a9603fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 15 Jan 2014 23:10:05 +0100
Subject: [PATCH] set mkspec path as set in meta-qt5
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [config]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 683a5b9..2cf079a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -111,7 +111,7 @@ QT.@PRI_TARGET_BASENAME@.depends = @PRI_TARGET_QTDEPS@"
 
 # This default value makes "installing qt and kf5 into the same prefix" work out of the box.
 # Packagers who use -DCMAKE_INSTALL_PREFIX=/usr will certainly want to set ECM_MKSPECS_INSTALL_DIR to something like share/qt5/mkspecs/modules
-set(ECM_MKSPECS_INSTALL_DIR mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
+set(ECM_MKSPECS_INSTALL_DIR lib/qt5/mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/qt_${PRI_TARGET_BASENAME}.pri DESTINATION ${ECM_MKSPECS_INSTALL_DIR})
 #### end of qmake support
-- 
1.8.3.1

