From c469a2111eea16edb70e847b71011afd08ac4a84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 29 Aug 2015 23:35:46 +0200
Subject: [PATCH 2/2] fix install path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e04d6af..cbd4950 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,4 +36,4 @@ target_link_libraries (${library_name}
    ${LIBGCRYPT_LIBRARIES}
 )
 
-install(TARGETS ${library_name} DESTINATION /${CMAKE_INSTALL_LIBDIR}/security)
+install(TARGETS ${library_name} DESTINATION ${CMAKE_INSTALL_LIBDIR}/security)
-- 
2.1.0

