From 599e78547a32ec3cd01ce9e007531b765f4168ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 2 Jul 2018 00:39:01 +0200
Subject: [PATCH] InOut/pmidi.c: Fix build with gcc8
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 InOut/pmidi.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/InOut/pmidi.c b/InOut/pmidi.c
index 8dfe0e71d..f6de618cf 100644
--- a/InOut/pmidi.c
+++ b/InOut/pmidi.c
@@ -188,14 +188,14 @@ static int listDevices(CSOUND *csound, CS_MIDIDEVICE *list, int isOutput){
   for (i = 0; i < cnt; i++) {
     info = portMidi_getDeviceInfo(i, isOutput);
     if (info->name != NULL)
-      strncpy(list[i].device_name, info->name, 63);
+      strncpy(list[i].device_name, info->name, 64);
     snprintf(tmp, 64, "%d", i);
-    strncpy(list[i].device_id, tmp, 63);
+    strncpy(list[i].device_id, tmp, 64);
     list[i].isOutput = isOutput;
     if (info->interf != NULL)
-      strncpy(list[i].interface_name, info->interf, 63);
+      strncpy(list[i].interface_name, info->interf, 64);
     else strcpy(list[i].interface_name, "");
-    strncpy(list[i].midi_module, drv, 63);
+    strncpy(list[i].midi_module, drv, 64);
   }
   return cnt;
 }
-- 
2.14.4

