From 3d5e27ea49f67b38a39ad831d18e8abd17b3a9bc Mon Sep 17 00:00:00 2001
From: vlazzarini <vlazzarini@mu.ie>
Date: Sun, 27 May 2018 11:18:08 +0100
Subject: [PATCH] fixed unsafe use of strncpy

Upstream-Status: Backport

---
 util1/scot/scot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util1/scot/scot.c b/util1/scot/scot.c
index 857bb08ce..5b9a0eb96 100644
--- a/util1/scot/scot.c
+++ b/util1/scot/scot.c
@@ -684,7 +684,7 @@ void addparam(int n,            /* number of param to change */
 
     ps = findparam(n, ptop);
     if (strcmp(s, "."))
-      strncpy(ps, s, 1+strlen(s));
+      strncpy(ps, s, 31);
 }
 
 static                          /* returns pointer to */
-- 
2.14.4

