From e74c303363ad3d09d0987c7d6b807479a23ef186 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 22 May 2020 14:38:40 +0200
Subject: [PATCH] FIND_REQUIRED_PROGRAM is broken - use the tools directly
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 cmake/FindRequiredProgram.cmake | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/cmake/FindRequiredProgram.cmake b/cmake/FindRequiredProgram.cmake
index 11af04e..673b981 100644
--- a/cmake/FindRequiredProgram.cmake
+++ b/cmake/FindRequiredProgram.cmake
@@ -26,14 +26,7 @@
 # usage: FIND_REQUIRED_PROGRAM(variable name1 [name2] ...)
 MACRO(FIND_REQUIRED_PROGRAM _variable)
 
-    FIND_PROGRAM(${_variable} NAMES ${ARGN})
-
-    IF (${_variable})
-        GET_FILENAME_COMPONENT(_basename ${${_variable}} NAME_WE)
-        MESSAGE(STATUS "Found ${_basename}: ${${_variable}}")
-    ELSE (${_variable})
-        MESSAGE(FATAL_ERROR "Unable to find executable for ${ARGN}")
-    ENDIF (${_variable})
+    SET(_variable, ${ARGN})
 
 ENDMACRO(FIND_REQUIRED_PROGRAM)
 
-- 
2.21.1

