From cb9bb7f24cd21fb0fa1742a4316ad489f6c8cf6d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 11 Oct 2015 21:24:16 +0200
Subject: [PATCH] fix build with QT_NO_SESSIONMANAGER set
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/main.cpp | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index b27de6f..556fb16 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -70,10 +70,14 @@ int main(int argc, char **argv)
         KgDifficultyLevel::VeryHard
     );
     KDBusService service;
+#ifndef QT_NO_SESSIONMANAGER
     // see if we are starting with session management
     if (app.isSessionRestored()) {
         RESTORE(MainWindow);
-    } else {
+    }
+    else
+#endif
+    {
         MainWindow *window = new MainWindow;
         window->show();
     }
-- 
2.1.0

