From 39b270db699edb25293c49fe8904a810534c48b1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 16 Dec 2015 21:03:20 +0100
Subject: [PATCH] avoid autotests
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0d1f96d..92eec98 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,7 +58,6 @@ ecm_setup_version(${KF5_VERSION}
     SOVERSION 5)
 
 add_subdirectory(src)
-add_subdirectory(autotests)
 add_subdirectory(tests)
 
 
-- 
2.5.0

