From 2999f6b8525cac503122d95197f7a3ae314db4e4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sun, 29 Jan 2017 21:13:45 +0100
Subject: [PATCH 3/3] do not create ttl files - it won't work fo cross
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 builds/linux/LV2/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/builds/linux/LV2/Makefile b/builds/linux/LV2/Makefile
index 4a9995f..8ec7d44 100644
--- a/builds/linux/LV2/Makefile
+++ b/builds/linux/LV2/Makefile
@@ -11,10 +11,8 @@ ttl_generator:
 	$(MAKE) -f Makefile.ttl_generator CONFIG=$(CONFIG)
 
 lv2: ttl_generator binary
-	./lv2_ttl_generator build/helm.so
 	mkdir -p helm.lv2
 	cp build/helm.so helm.lv2
-	mv *.ttl helm.lv2
 
 clean:
 	$(MAKE) clean CONFIG=$(CONFIG) -f Makefile.binary
