From b6937b83618cee3e88c42071a5353fcb684d31bb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 2 Aug 2017 00:06:26 +0200
Subject: [PATCH] use standard vst path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b652a8b..61e223b 100644
--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@ PROGRAM = helm
 BIN     = $(DESTDIR)/usr/bin
 BINFILE = $(BIN)/$(PROGRAM)
 LV2     = $(DESTDIR)/$(LIBDIR)/lv2/$(PROGRAM).lv2
-VSTDIR  = $(DESTDIR)/$(LIBDIR)/lxvst
+VSTDIR  = $(DESTDIR)/$(LIBDIR)/vst
 VST     = $(VSTDIR)/$(PROGRAM).so
 SYSDATA = $(DESTDIR)/usr/share/$(PROGRAM)
 IMAGES  = $(SYSDATA)/icons
-- 
2.9.4

