From 760e76e826f935c72b4874f104773c954def532e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 24 Apr 2021 15:57:57 +0200
Subject: [PATCH 4/4] don't build documentation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a12d99a..75b9c86 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,12 +100,7 @@ ecm_set_disabled_deprecation_versions(
     KF 5.95
 )
 
-if (KF5I18n_FOUND)
-    ki18n_install(po)
-    kdoctools_install(po)
-endif()
 add_subdirectory(common)
-add_subdirectory(docs)
 add_subdirectory(src)
 if (BUILD_TESTING)
     add_subdirectory(autotests)
-- 
2.35.3

