From c4030d7fec972b950c02668f4c32a1e412be286e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 28 Nov 2014 10:04:17 +0100
Subject: [PATCH] Add cmdline param to help find xsl
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

---
 CMakeLists.txt             | 2 ++
 cmake/FindDocBookXSL.cmake | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d99537c..a06d025 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,6 +30,8 @@ option(INSTALL_INTERNAL_TOOLS "Install internal tools too." OFF)
 option(BUILD_QCH "Build API documentation in QCH format (for e.g. Qt Assistant, Qt Creator & KDevelop)" OFF)
 add_feature_info(QCH ${BUILD_QCH} "API documentation in QCH format (for e.g. Qt Assistant, Qt Creator & KDevelop)")
 
+# OE specific
+set(OE_ROOT_PATH_PREFIX "" CACHE STRING "OE sysroot")
 # Dependencies
 set(REQUIRED_QT_VERSION 5.10.0)
 find_package(Qt5Core ${REQUIRED_QT_VERSION} REQUIRED NO_MODULE)
diff --git a/cmake/FindDocBookXSL.cmake b/cmake/FindDocBookXSL.cmake
index 3d9e40c..779cb2c 100644
--- a/cmake/FindDocBookXSL.cmake
+++ b/cmake/FindDocBookXSL.cmake
@@ -56,7 +56,7 @@ find_path (DocBookXSL_DIR lib/lib.xsl
 if (NOT DocBookXSL_DIR)
     # hacks for systems that put the version in the stylesheet dirs
     set (STYLESHEET_PATH_LIST)
-    foreach (STYLESHEET_PREFIX_ITER ${CMAKE_SYSTEM_PREFIX_PATH})
+    foreach (STYLESHEET_PREFIX_ITER ${CMAKE_SYSTEM_PREFIX_PATH} ${OE_ROOT_PATH_PREFIX})
         file(GLOB STYLESHEET_SUFFIX_ITER RELATIVE ${STYLESHEET_PREFIX_ITER}
             ${STYLESHEET_PREFIX_ITER}/share/xml/docbook/xsl-stylesheets-*
         )
