From 1f7e6c3b0bbbb25e83550739692e857fc5d7ea84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 5 Jan 2015 13:39:45 +0100
Subject: [PATCH 2/2] CMakeLists.txt: do not search for libintl / gettext
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt            | 6 ------
 cmake/KF5I18NMacros.cmake | 1 -
 2 files changed, 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 875ceb4..a50e083 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,12 +39,6 @@ include(KDEInstallDirs)
 include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
 include(KDECMakeSettings)
 
-find_package(LibIntl)
-set_package_properties(LibIntl PROPERTIES TYPE REQUIRED
-	URL "http://gnuwin32.sourceforge.net/packages/libintl.htm"
-	PURPOSE "Needed for building KI18n unless glibc is the system libc implementation"
-)
-
 # usually is called using find_package(KF5I18n),
 # KF5I18NMacros.cmake needs to know the scripts directory
 set(KF5I18n_DIR ${CMAKE_CURRENT_LIST_DIR}/cmake)
diff --git a/cmake/KF5I18NMacros.cmake b/cmake/KF5I18NMacros.cmake
index 022de52..95ce68a 100644
--- a/cmake/KF5I18NMacros.cmake
+++ b/cmake/KF5I18NMacros.cmake
@@ -25,7 +25,6 @@
 # OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 # SUCH DAMAGE.
 
-find_package(Gettext REQUIRED)
 find_package(PythonInterp REQUIRED)
 
 set(_ki18n_pmap_compile_script ${CMAKE_CURRENT_LIST_DIR}/ts-pmap-compile.py)
-- 
2.9.3

