From 69bf91f00ec2f62907d57e6ce0f1a044669ba837 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 13 Jun 2021 22:58:03 +0200
Subject: [PATCH] do not install po-files for native build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>

---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 95b197f..85b790a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,7 +88,6 @@ ecm_set_disabled_deprecation_versions(
     QT 5.15.2
 )
 
-ki18n_install(po)
 add_subdirectory(src)
 if (BUILD_TESTING)
     add_subdirectory(autotests)
-- 
2.31.1

