From 19e25a85ac0c35653546be6633502941768a5206 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 3 Aug 2018 11:30:05 +0200
Subject: [PATCH] Do not hardcode python path - it points to sysroot
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 cmake/KF5I18NMacros.cmake.in | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/cmake/KF5I18NMacros.cmake.in b/cmake/KF5I18NMacros.cmake.in
index e8401a9..0bfc2a1 100644
--- a/cmake/KF5I18NMacros.cmake.in
+++ b/cmake/KF5I18NMacros.cmake.in
@@ -27,8 +27,6 @@
 
 find_package(Gettext REQUIRED)
 
-set(KI18N_PYTHON_EXECUTABLE "@PYTHON_EXECUTABLE@")
-
 set(_ki18n_pmap_compile_script ${CMAKE_CURRENT_LIST_DIR}/ts-pmap-compile.py)
 set(_ki18n_uic_script ${CMAKE_CURRENT_LIST_DIR}/kf5i18nuic.cmake)
 
@@ -116,7 +114,7 @@ function(KI18N_INSTALL podir)
     add_custom_target(tsfiles-${pathmd5} ALL
         COMMENT "Generating ts..."
         COMMAND ${CMAKE_COMMAND}
-                -DPYTHON_EXECUTABLE=${KI18N_PYTHON_EXECUTABLE}
+                -DPYTHON_EXECUTABLE=python
                 -D_ki18n_pmap_compile_script=${_ki18n_pmap_compile_script}
                 -DCOPY_TO=${CMAKE_CURRENT_BINARY_DIR}/${dirname}
                 -DPO_DIR=${absolute_podir}
@@ -158,7 +156,7 @@ function(KI18N_INSTALL_TS_FILES lang scripts_dir)
             set(pmapc_basename "${pmap_basename}c")
             set(pmapc_file "${lang}-${subpath}-${pmapc_basename}")
             add_custom_command(OUTPUT ${pmapc_file}
-               COMMAND ${KI18N_PYTHON_EXECUTABLE}
+               COMMAND python
                ARGS
                -B
                ${_ki18n_pmap_compile_script}
-- 
2.14.4

