From 99c66465200cbbc5849349305ccc61c9285ea2d7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 15 Oct 2018 22:23:58 +0200
Subject: [PATCH] Do not hardcode python path - it points to sysroot
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 cmake/KF5I18nMacros.cmake.in | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/cmake/KF5I18nMacros.cmake.in b/cmake/KF5I18nMacros.cmake.in
index 3824acb..73387f2 100644
--- a/cmake/KF5I18nMacros.cmake.in
+++ b/cmake/KF5I18nMacros.cmake.in
@@ -27,10 +27,6 @@
 
 find_package(Gettext REQUIRED)
 
-# The Python executable used for building ki18n will be used as a fallback
-# solution if it cannot be found in $PATH when building applications.
-set(_KI18N_PYTHON_EXECUTABLE "@PYTHON_EXECUTABLE@")
-
 find_program(KI18N_PYTHON_EXECUTABLE NAMES python3 python2 python)
 if(NOT KI18N_PYTHON_EXECUTABLE)
     set(KI18N_PYTHON_EXECUTABLE "${_KI18N_PYTHON_EXECUTABLE}")
@@ -138,7 +134,7 @@ function(KI18N_INSTALL podir)
     add_custom_target(tsfiles-${pathmd5} ALL
         COMMENT "Generating ts..."
         COMMAND ${CMAKE_COMMAND}
-                -DPYTHON_EXECUTABLE=${KI18N_PYTHON_EXECUTABLE}
+                -DPYTHON_EXECUTABLE=python3
                 -D_ki18n_pmap_compile_script=${_ki18n_pmap_compile_script}
                 -DCOPY_TO=${CMAKE_CURRENT_BINARY_DIR}/${dirname}
                 -DPO_DIR=${absolute_podir}
@@ -180,7 +176,7 @@ function(KI18N_INSTALL_TS_FILES lang scripts_dir)
             set(pmapc_basename "${pmap_basename}c")
             set(pmapc_file "${lang}-${subpath}-${pmapc_basename}")
             add_custom_command(OUTPUT ${pmapc_file}
-               COMMAND ${KI18N_PYTHON_EXECUTABLE}
+               COMMAND python3
                ARGS
                -B
                ${_ki18n_pmap_compile_script}
-- 
2.14.4

