From 09db90cefb2d466e143ee655e3a10220d86fb88b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 5 Mar 2016 22:28:58 +0100
Subject: [PATCH 3/3] align binary installation folders
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 app/app.pro | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/app/app.pro b/app/app.pro
index 5559ab2..e425c55 100644
--- a/app/app.pro
+++ b/app/app.pro
@@ -39,11 +39,11 @@ linux {
     # Rename file from "Antimony" to "antimony"
     QMAKE_POST_LINK += $(MOVE) $$OUT_PWD/$${TARGET} $$OUT_PWD/$$lower($${TARGET})
 
-    executable.path = /usr/local/bin
+    executable.path = /usr/bin
     executable.files = antimony
-    nodes_folder.path = /usr/local/bin/sb/nodes
+    nodes_folder.path = /usr/bin/sb/nodes
     nodes_folder.files = ../py/nodes/*
-    fab_folder.path = /usr/local/bin/sb/fab
+    fab_folder.path = /usr/bin/sb/fab
     fab_folder.files = ../py/fab/*
     INSTALLS += executable nodes_folder fab_folder
 }
-- 
2.5.0

