From 23dd312b270330a0df18b229b9f68bc74c72934b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Thu, 1 Nov 2018 20:31:34 +0100
Subject: [PATCH] Align Makefiles
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 apps/Makefile   | 11 +++--------
 source/Makefile |  9 ++-------
 2 files changed, 5 insertions(+), 15 deletions(-)

diff --git a/apps/Makefile b/apps/Makefile
index 053b9a6..2becb79 100644
--- a/apps/Makefile
+++ b/apps/Makefile
@@ -18,14 +18,9 @@
 # ----------------------------------------------------------------------------
 
 
-PREFIX ?= /usr/local
-BINDIR ?= $(PREFIX)/bin
-MANDIR ?= /usr/share/man/man1
-
 VERSION = 1.6.0
 CPPFLAGS += -MMD -MP -DVERSION=\"$(VERSION)\"
-CXXFLAGS += -O2  -ffast-math -Wall
-CXXFLAGS += -march=native
+CXXFLAGS += -ffast-math -Wall
 
 
 all:	zresample zretune zresample.1.gz zretune.1.gz
@@ -55,8 +50,8 @@ zretune.1.gz:	zretune.1
 
 
 install:	all
-	install -d $(BINDIR)
-	install -d $(MANDIR)
+	install -d $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)
 	install -m 755 zresample $(DESTDIR)$(BINDIR)
 	install -m 755 zretune $(DESTDIR)$(BINDIR)
 	install -m 644 zresample.1.gz $(DESTDIR)$(MANDIR)
diff --git a/source/Makefile b/source/Makefile
index cd54ff5..ccb68d1 100644
--- a/source/Makefile
+++ b/source/Makefile
@@ -20,10 +20,6 @@
 
 # Modify as required.
 #
-SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
-PREFIX ?= /usr/local
-INCDIR ?= $(PREFIX)/include
-LIBDIR ?= $(PREFIX)/lib$(SUFFIX)
 
 MAJVERS = 1
 MINVERS = 6.2
@@ -32,8 +28,7 @@ DISTDIR = zita-resampler-$(VERSION)
 
 
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
-CXXFLAGS += -Wall -fPIC -O2 -ffast-math
-CXXFLAGS += -march=native
+CXXFLAGS += -Wall -fPIC -ffast-math
 LDFLAGS += 
 LDLIBS +=
 
@@ -58,7 +53,7 @@ install:	$(ZITA-RESAMPLER_MIN)
 	install -d $(DESTDIR)$(LIBDIR)
 	install -m 644 $(ZITA-RESAMPLER_H)   $(DESTDIR)$(INCDIR)/zita-resampler
 	install -m 755 $(ZITA-RESAMPLER_MIN) $(DESTDIR)$(LIBDIR)
-	ldconfig
+	ldconfig -n $(DESTDIR)$(LIBDIR)
 	ln -sf $(ZITA-RESAMPLER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-RESAMPLER_SO)
 
 uninstall:
-- 
2.14.4

