From b54684b9e48bbe6783dd3c690524ed2d21dc30ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Tue, 5 May 2015 14:29:15 +0200
Subject: [PATCH] fix build for qtbase build without session support
 QT_NO_SESSIONMANAGER
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 main.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index ec62c67..fd0c921 100644
--- a/main.cpp
+++ b/main.cpp
@@ -89,9 +89,12 @@ int main(int argc, char **argv)
     }
     else
     {
+#ifndef QT_NO_SESSIONMANAGER
         if ( app.isSessionRestored() )
             RESTORE(AtomTopLevel)
-        else {
+        else
+#endif
+        {
             AtomTopLevel *top = new AtomTopLevel;
             if ( parser.isSet( QStringLiteral("hackmode") ) )
                 top->enableHackMode();
-- 
2.5.5

