From 3c01957dd2640c8175efe57ad517fe41e265b6ab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 27 Nov 2014 18:11:20 +0100
Subject: [PATCH] debug
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 cmake/FindDocBookXSL.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/FindDocBookXSL.cmake b/cmake/FindDocBookXSL.cmake
index 6429e90..4d9c600 100644
--- a/cmake/FindDocBookXSL.cmake
+++ b/cmake/FindDocBookXSL.cmake
@@ -34,6 +34,7 @@ if (NOT DocBookXSL_DIR)
     # hacks for systems that put the version in the stylesheet dirs
     set (STYLESHEET_PATH_LIST)
     foreach (STYLESHEET_PREFIX_ITER ${CMAKE_SYSTEM_PREFIX_PATH})
+        message(WARNING ${STYLESHEET_PREFIX_ITER})
         file(GLOB STYLESHEET_SUFFIX_ITER RELATIVE ${STYLESHEET_PREFIX_ITER}
             ${STYLESHEET_PREFIX_ITER}/share/xml/docbook/xsl-stylesheets-*
         )
-- 
1.8.3.1

