From d9a64eaffdca73e50b767f58c87f4299db85cd83 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Fri, 28 Nov 2014 10:28:02 +0100
Subject: [PATCH 2/4] build meinproc5 - it works fine in our environment
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 341ecf4..a440a1b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -47,11 +47,6 @@ install(TARGETS KF5XsltKde EXPORT KF5DocToolsTargets ${KF5_INSTALL_TARGETS_DEFAU
 
 ########### next target ###############
 
-# The cross compiling parts are commented out on purpose. Alex
-if (CMAKE_CROSSCOMPILING)
-#    set(IMPORT_MEINPROC5_EXECUTABLE "${KDE_HOST_TOOLS_PATH}/ImportMeinProc5Executable.cmake" CACHE FILEPATH "Point it to the export file of meinproc5 from a native build")
-#    include(${IMPORT_MEINPROC5_EXECUTABLE})
-else ()
     set_source_files_properties(xslt.cpp PROPERTIES COMPILE_FLAGS -DSIMPLE_XSLT )
 
     option(MEINPROC_NO_KARCHIVE "Compile meinproc without KArchive dependency" OFF)
@@ -71,7 +66,6 @@ else ()
 
 #    # export this binary for cross-compilation
 #    export(TARGETS meinproc5 FILE ${CMAKE_BINARY_DIR}/ImportMeinProc4Executable.cmake)
-endif ()
 
 ########### customize includes for xslt ###############
 # WARNING: this has severe limitations on the windows platform since the .xsl
-- 
1.8.3.1

