From 22ce88122c616771e715873cc715b1ded65f0b52 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Tue, 14 Nov 2017 14:11:19 +0100
Subject: [PATCH] do not check for xdg-udser-dirs at build time - it is a
 runtime dependency
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3cd4112..a400535 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,9 +26,6 @@ find_package(KF5WindowSystem REQUIRED)
 
 find_package(lxqt REQUIRED)
 
-# Please don't move, must be after lxqt
-find_package(XdgUserDirs REQUIRED)
-
 if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release)
 endif()
-- 
2.9.5

