From 36b834661ac4169db20c682bddfdcdf96df5a2f7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 19 Jan 2019 00:44:13 +0100
Subject: [PATCH] Do not inject /usr/local/include to include paths
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 waflib/extras/autowaf.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/waflib/extras/autowaf.py b/waflib/extras/autowaf.py
index feaae3c6..119c95b5 100644
--- a/waflib/extras/autowaf.py
+++ b/waflib/extras/autowaf.py
@@ -189,8 +189,6 @@ def check_pkg(conf, name, **args):
             if 'COMPILER_CXX' in conf.env:
                 conf.env.append_value('CXXFLAGS', ['-isystem', path])
 
-        conf.env.append_value('CXXFLAGS', ['-isystem', '/usr/local/include'])
-
 def normpath(path):
     if sys.platform == 'win32':
         return os.path.normpath(path).replace('\\', '/')
-- 
2.20.1

