From 3de9eb59c0fe39c4a4ecb182b3bc7f53981024b5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 12 Jun 2020 21:19:00 +0200
Subject: [PATCH] Fix build with Qt 5.15
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Submitted[https://github.com/lirios/materialdecoration/pull/9]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 src/plugins/decorations/material/materialdecoration.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/decorations/material/materialdecoration.cpp b/src/plugins/decorations/material/materialdecoration.cpp
index 73073f9..3a4a9b6 100644
--- a/src/plugins/decorations/material/materialdecoration.cpp
+++ b/src/plugins/decorations/material/materialdecoration.cpp
@@ -31,6 +31,7 @@
  ***************************************************************************/
 
 #include <QtMath>
+#include <QPainterPath>
 
 #include "materialdecoration.h"
 
-- 
2.26.2

