From e2067d92e3f962a6efa51e7a5902b64f4d58f949 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Wed, 10 Dec 2014 08:11:46 +0100
Subject: [PATCH] build only native katehighlightingindexer
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 CMakeLists.txt | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9746c7a..798b0a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,13 +34,6 @@ set(REQUIRED_QT_VERSION "5.2.0")
 # Required Qt5 components to build this framework
 find_package(Qt5 ${REQUIRED_QT_VERSION} NO_MODULE REQUIRED Core Widgets Script PrintSupport Xml XmlPatterns)
 
-find_package(KF5Archive ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5Config ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5GuiAddons ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5I18n ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5KIO ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5Parts ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5Sonnet ${KF5_DEP_VERSION} REQUIRED)
 
 # vi mode on per default
 option (BUILD_VIMODE "Build vimode in" ON)
@@ -50,7 +43,7 @@ add_definitions(-DTRANSLATION_DOMAIN=\"ktexteditor5\")
 if (IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
     ki18n_install(po)
 endif()
-add_subdirectory(src)
+add_subdirectory(src/syntax/data)
 if (BUILD_TESTING)
     add_subdirectory(autotests)
 endif()
@@ -71,13 +64,6 @@ install(FILES
   COMPONENT Devel
 )
 
-install(EXPORT
-  KF5TextEditorTargets
-  DESTINATION "${CMAKECONFIG_INSTALL_DIR}"
-  FILE KF5TextEditorTargets.cmake
-  NAMESPACE KF5::
-)
-
 install(FILES
   "${CMAKE_CURRENT_BINARY_DIR}/ktexteditor_version.h"
   DESTINATION "${KF5_INCLUDE_INSTALL_DIR}"
-- 
1.8.3.1

