From 95d769946706abf85f817396996f5bff6c9ac18e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 12 Jul 2021 22:15:51 +0200
Subject: [PATCH] Fix EGL link
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 src/declarativeimports/core/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/declarativeimports/core/CMakeLists.txt b/src/declarativeimports/core/CMakeLists.txt
index df00239..d6be6c2 100644
--- a/src/declarativeimports/core/CMakeLists.txt
+++ b/src/declarativeimports/core/CMakeLists.txt
@@ -58,7 +58,7 @@ if(HAVE_X11)
 endif()
 
 if(HAVE_EGL)
-    target_link_libraries(corebindingsplugin ${EGL_TARGET})
+    target_link_libraries(corebindingsplugin ${OPENGL_gl_LIBRARY} ${EGL_TARGET})
 else()
 
 endif()
-- 
2.31.1

