From ead044b5696b11f83d4728908d6d47a55450a9ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 9 Feb 2015 14:46:09 +0100
Subject: [PATCH] fix path to khc_indexbuilder
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 config-khelpcenter.h.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config-khelpcenter.h.cmake b/config-khelpcenter.h.cmake
index 2a4839d..5d90483 100644
--- a/config-khelpcenter.h.cmake
+++ b/config-khelpcenter.h.cmake
@@ -1 +1 @@
-#define KHC_INDEXBUILDER "@CMAKE_INSTALL_PREFIX@/@LIBEXEC_INSTALL_DIR@/khc_indexbuilder"
+#define KHC_INDEXBUILDER "@LIBEXEC_INSTALL_DIR@/khc_indexbuilder"
-- 
1.9.3

