From 7c4d3532bd5540d403f4d477beb473f0948c9e32 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Thu, 5 Jan 2017 11:45:38 +0100
Subject: [PATCH 3/3] dssi: fix preset selection
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/dssi.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/dssi.cpp b/src/dssi.cpp
index 0d5528c..f063a64 100644
--- a/src/dssi.cpp
+++ b/src/dssi.cpp
@@ -137,7 +137,10 @@ static void select_program(LADSPA_Handle Instance, unsigned long Bank, unsigned
 
 	TRACE_ARGS("Bank = %d Index = %d", Bank, Index);
 
-	if (Bank == 0 && Index < 128) {
+	const std::vector<BankInfo> banks = PresetController::getPresetBanks();
+
+	if (Bank < banks.size() && Index < PresetController::kNumPresets) {
+		s_presetController->loadPresets(banks[Bank].file_path.c_str());
 		a->synth->setPresetNumber(Index);
 		// now update DSSI host's view of the parameters
 		for (unsigned i = 0; i < kAmsynthParameterCount; i++) {
-- 
2.5.5

