From 8a09ea3ea3be54e200480dc03e375aaae2bba4ef Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Fri, 2 Nov 2018 13:51:51 +0100
Subject: [PATCH] Do not install to /usr/local
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 qskconfig.pri | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/qskconfig.pri b/qskconfig.pri
index f78c0c4..1926b5e 100644
--- a/qskconfig.pri
+++ b/qskconfig.pri
@@ -10,10 +10,6 @@ QSK_VERSION      = $${QSK_VER_MAJ}.$${QSK_VER_MIN}.$${QSK_VER_PAT}
 
 QSK_INSTALL_PREFIX    = $$[QT_INSTALL_PREFIX]
 
-unix {
-    QSK_INSTALL_PREFIX    = /usr/local/qskinny-$${QSK_VERSION}
-}
-
 win32 {
     QSK_INSTALL_PREFIX    = C:/Qskinny-$${QSK_VERSION}
 }
-- 
2.14.4

