From 45038115cf962f0b5923a74ae362e3a430ec5a3b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Sat, 21 Feb 2015 16:35:40 +0100
Subject: [PATCH] fix qml install dir
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 components/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/components/CMakeLists.txt b/components/CMakeLists.txt
index bf95f4c..ac75b90 100644
--- a/components/CMakeLists.txt
+++ b/components/CMakeLists.txt
@@ -1,8 +1,8 @@
 configure_file("${COMPONENTS_VERSION}/LayoutBox.qml" "${COMPONENTS_VERSION}/LayoutBox.qml")
 configure_file("common/qmldir" "common/qmldir")
 
-install(DIRECTORY "${COMPONENTS_VERSION}/" DESTINATION "${QT_IMPORTS_DIR}/SddmComponents")
-install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${COMPONENTS_VERSION}/" DESTINATION "${QT_IMPORTS_DIR}/SddmComponents")
+install(DIRECTORY "${COMPONENTS_VERSION}/" DESTINATION "${QML_INSTALL_DIR}/SddmComponents")
+install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${COMPONENTS_VERSION}/" DESTINATION "${QML_INSTALL_DIR}/SddmComponents")
 
-install(DIRECTORY "common/" DESTINATION "${QT_IMPORTS_DIR}/SddmComponents")
-install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/common/" DESTINATION "${QT_IMPORTS_DIR}/SddmComponents")
+install(DIRECTORY "common/" DESTINATION "${QML_INSTALL_DIR}/SddmComponents")
+install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/common/" DESTINATION "${QML_INSTALL_DIR}/SddmComponents")
-- 
1.9.3

