From dd670c0e3dd3f0a7358740d8e198686cc2826beb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 13 Oct 2018 00:01:01 +0200
Subject: [PATCH] Make itstool work
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [cross specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 gtk2_ardour/wscript |  4 ++--
 wscript             | 13 -------------
 2 files changed, 2 insertions(+), 15 deletions(-)

diff --git a/gtk2_ardour/wscript b/gtk2_ardour/wscript
index d98e4e991..7a872382b 100644
--- a/gtk2_ardour/wscript
+++ b/gtk2_ardour/wscript
@@ -957,7 +957,7 @@ def appdata_i18n_pot(bld):
                 'ardour.appdata.xml.in.in')
         Logs.info('Updating ' + potfile)
         try:
-                subprocess.call(('itstool', '-i', itsfile, '-o', potfile, adsource))
+                subprocess.call(('python3', '%sysroot_bindir%/itstool', '-i', itsfile, '-o', potfile, adsource))
         except:
                 print ('Error processing appdata pot file - ignored')
     elif bld.cmd == 'clean' and os.path.exists(potfile):
@@ -982,7 +982,7 @@ def appdata_i18n_xmlin(bld):
         mo_glob = os.path.join(bld.top_dir, 'gtk2_ardour', 'appdata', 'po', '*.mo')
         mo_files = glob.glob(mo_glob)
         Logs.info('Generating ' + adxmlinfile)
-        subprocess.call(['itstool', '-i', itsfile, '-o', adxmlinfile,
+        subprocess.call(['python3', '%sysroot_bindir%/itstool', '-i', itsfile, '-o', adxmlinfile,
             '-j', adxmlininfile] + mo_files)
     elif bld.cmd == 'clean' and os.path.exists(adxmlinfile):
         Logs.info('Removing ' + adxmlinfile)
diff --git a/wscript b/wscript
index 6bdef92f6..1e3552f61 100644
--- a/wscript
+++ b/wscript
@@ -861,19 +861,6 @@ def configure(conf):
         conf.env['CFLAGS'] = []
         conf.env['CXXFLAGS'] = []
 
-    # freedesktop translations needs itstool > 1.0.3 (-j option)
-    if Options.options.freedesktop:
-        output = subprocess.Popen("itstool --version", shell=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE).communicate()[0].splitlines()
-        o = output[0].decode('utf-8')
-        itstool = o.split(' ')[0]
-        version = o.split(' ')[1].split('.')
-        # use  distutils.version.StrictVersion  or something python to check >= 1.0.4
-        # but first make sure that all build-hosts (incl. OSX-10.5/PPC) have that python lib.
-        # lazy approach: just use major version 2.X.X
-        if itstool != "itstool" or version[0] < "2":
-            print("--freedesktop requires itstool > 2.0.0 to translate files.")
-            sys.exit(-1)
-
     conf.env['VERSION'] = VERSION
     conf.env['MAJOR'] = MAJOR
     conf.env['MINOR'] = MINOR
-- 
2.14.4

