From 7e2344053441647b6ece33fb6076b6ab302ba144 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 9 Oct 2022 16:11:25 +0200
Subject: [PATCH] Use native mktags
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 Makefile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.in b/Makefile.in
index 085fbd5..b73f301 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -93,7 +93,7 @@ VERSION:
 tags.o: tags.c cstring.h tags.h blocktags
 
 blocktags: mktags
-	./mktags > blocktags
+	mktags > blocktags
 
 mktags: mktags.o
 	$(LINK) -o mktags mktags.o
-- 
2.35.3

