From 4fa9e08ddca36fc3cebd35b6f0ea2fc344b85e13 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sun, 28 Oct 2018 21:49:42 +0100
Subject: [PATCH] Make QML installation path configurable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 837c6bb..cc9be3c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,7 @@ option(QTGSTREAMER_EXAMPLES "Build QtGStreamer's examples" ON)
 option(QTGSTREAMER_CODEGEN "Build and use QtGStreamer's codegen" OFF)
 option(USE_GST_PLUGIN_DIR "Install gstreamer plugins at the system location" ON)
 option(USE_QT_PLUGIN_DIR "Install qt plugins at the system location" ON)
+set(QMLDIR "/usr/lib/qml" CACHE STRING "QMLpath")
 
 include(GNUInstallDirs)
 include(MacroLogFeature)
@@ -165,7 +166,7 @@ endif()
 
 if (USE_QT_PLUGIN_DIR)
     set(QTGSTREAMER_QTQUICK1_INSTALL_DIR ${QT_IMPORTS_DIR})
-    set(QTGSTREAMER_QTQUICK2_INSTALL_DIR ${QT_QML_DIR})
+    set(QTGSTREAMER_QTQUICK2_INSTALL_DIR ${QMLDIR})
 else()
     if (USE_QT5)
         set(QTGSTREAMER_QTQUICK1_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/qt5/imports)
-- 
2.14.4

