From b37875b04823545b863c126f97b1558b92763584 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@googlemail.com>
Date: Mon, 24 Oct 2016 00:58:47 +0200
Subject: [PATCH] projectM-test: add missing build paths
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Pending

Signed-off-by: Andreas Müller <schnitzeltony@googlemail.com>
---
 src/projectM-test/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/projectM-test/CMakeLists.txt b/src/projectM-test/CMakeLists.txt
index bbd1e51..4121049 100644
--- a/src/projectM-test/CMakeLists.txt
+++ b/src/projectM-test/CMakeLists.txt
@@ -35,11 +35,11 @@ elseif (${CMAKE_PROJECT_NAME} MATCHES ${PROJECT_NAME})
    set(PROJECTM_LINK ${LIBPROJECTM_LDFLAGS})
 endif (${CMAKE_PROJECT_NAME} MATCHES "projectM-complete")
 
-INCLUDE_DIRECTORIES(${PROJECTM_INCLUDE} ${SDL_INCLUDE_DIR} ${GTK_INCLUDE_DIR})
+INCLUDE_DIRECTORIES(${PROJECTM_INCLUDE} ${SDL_INCLUDE_DIR} ${GTK_INCLUDE_DIR} ${OPENGL_INCLUDE_DIR})
 LINK_DIRECTORIES(${PROJECTM_LINK})
 
-TARGET_LINK_LIBRARIES(projectM-test projectM  ${SDL_LIBRARY})
-TARGET_LINK_LIBRARIES(projectM-test-memleak projectM  ${SDL_LIBRARY} )
-TARGET_LINK_LIBRARIES(projectM-test-texture projectM  ${SDL_LIBRARY} )
+TARGET_LINK_LIBRARIES(projectM-test projectM  ${SDL_LIBRARY} ${OPENGL_gl_LIBRARY} )
+TARGET_LINK_LIBRARIES(projectM-test-memleak projectM  ${SDL_LIBRARY} ${OPENGL_gl_LIBRARY} )
+TARGET_LINK_LIBRARIES(projectM-test-texture projectM  ${SDL_LIBRARY} ${OPENGL_gl_LIBRARY} )
 
 INSTALL(TARGETS projectM-test projectM-test-texture DESTINATION ${CMAKE_INSTALL_PREFIX}/bin )
-- 
2.5.5

